\documentclass[12pt]{article}
\usepackage{geometry} 
\usepackage{graphicx}
\usepackage{float}
\usepackage{subcaption}
\usepackage{url}
\usepackage{mathtools}
\usepackage{color}
\usepackage{natbib}
\usepackage{fullpage}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{bigints}

% Make paragraphs not indented and leave a line between paragraphs
\setlength{\parskip}{\baselineskip}%
\setlength{\parindent}{0pt}%

\newcommand{\e}[1]{{\mathbb E}\left[ #1 \right]}
\newcommand{\admixsource}[1]{{$G^*$}}
\newcommand{\kadmixsource}[1]{{$G^{*}_{#1}$}}
\newcommand{\identifyadmixsource}[1]{{#1^{*}}}
\newcommand{\gb}[1]{{\em \color{magenta} #1}}
\newcommand{\plr}[1]{{\em \color{green} #1}}
\newcommand{\gc}[1]{{\em \color{blue} #1}}

\geometry{a4paper}

\title{A Novel Spatial Framework for Understanding Genetic Admixture}
\date{\vspace{-5ex}}
\author{Gideon S. Bradburd$^{1,a}$, Peter L. Ralph$^{3,b}$, Graham M. Coop$^{1,c}$}

\begin{document}

\maketitle

\textsuperscript{1}Center for Population Biology, Department of Evolution and Ecology, University of California, Davis, CA 95616

\textsuperscript{3}Department of Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089

\textsuperscript{a}gbradburd@ucdavis.edu; 
\textsuperscript{b}pralph@usc.edu;
\textsuperscript{c}gmcoop@ucdavis.edu\\\\\

\newpage

\begin{abstract}
The patterns of genetic variation observed in modern populations are the product of a complex demographic and evolutionary history.  Genetic data can be used to illuminate that history, providing information about when and how populations have diverged, how migration connects populations, and how population sizes have fluctuated over time.  Work in this area has largely focused on estimating a population phylogeny, in which shared branch length on a tree represents shared evolutionary history between a pair of populations sampled in the modern day.  However, patterns of population differentiation are rarely tree-like, as migration and colonization will continuously re-shape patterns of relatedness between populations.  Isolation by distance (IBD), in which population differentiation increases with the distance between them, may offer a more natural null hypothesis.  Here, we present a novel analytical framework, SpaceMix, for the study of spatial genetic variation and genetic admixture, and a simple statistic to describe a population’s admixture status.
\end{abstract}

\newpage
%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Introduction}

Population-level demographic processes leave their mark on patterns of genetic variation and differentiation within and between populations.  Recent population genetics methods have focused on ways of learning about those processes from genetic observations, including demographic inference from the allele frequency spectrum (Song) and inference of the ancestral recombination graph from orthologous sequence data (Song, Rasmussen\&Siepel).  

Further work has focused on joint inference and visualization of whole-genome patterns of relatedness, specifically on estimating a population phylogeny describing the evolutionary history shared between sampled individuals or populations (e.g. Pickrell \& Pritchard, Patterson \& Moorjani, Reich?).  The approach of modeling relatedness between populations within species as a tree-like structure was pioneered by Cavalli-Sforza (\&Thompson?) in the 1960s \gb{?}, when genetic character data (blood types) were first becoming available in humans.  

Their work has been greatly extended by Pickrell and Pritchard (2012), whose method, TreeMix, models covariance in allele frequencies across loci as a directed acyclic graph between populations.  TreeMix then accommodates reticulate population structure by allowing branches on the population tree tree to be connected by arrows of admixture that explain excess residual covariance between populations or population 'clades.' 

In parallel, Patterson and Reich and Moorjani and Other Folks have developed a suite of tests for genetic admixture between a set of populations assuming a tree-like structure to their evolutionary history.  These tests model shared genetic drift as shared branch length on a hypothesized population tree, and take covariance in excess of that stipulated by the tree to be evidence for admixture.

These tree-based methods are both valuable as genetic inference and visualization tools.  However, a more natural framework for modeling genetic differentiation between sampled populations or individuals may be that of isolation by distance (IBD) (Wright).  The pattern of IBD, in which genetic differentiation increases with the geographic distance between populations, is ubiquitous in nature (Meirmans 2012, Ralph \& Coop 2012), and relies only on the weak assumption that individual's mating opportunities are geographically limited by dispersal.  

\gb{transition section on visualization of population structure and patterns of relatedness}

There is a rich history of this type of data visualization, especially using reduced dimensionality representations of the data such as Principal Components Analysis (e.g. Menozzi, Piazza, and Cavalli-Sforza 1978, Novembre et al 2008, Novembre and Stephens 2008).  Our method offers an improvement over PCA-based visualization methods in that it offers an explicit model of spatial genetic variation, and can therefore be used as a robust framework for inference and hypothesis testing (see also Yang et al 2012, Yang et al 2014).

%Isolation by distance is, in many ways, a more reasonable null hypothesis of population relatedness than a population phylogeny;  \gb{too compare-y?  should I ditch this?} population structure will only rarely be truly tree-like, and a strictly bifurcating graph is unable to accommodate many geographic scenarios, such as multiple equidistant populations in migration-drift equilibrium.

Here, we present an analytical framework and develop an inference algorithm for studying the spatial distribution of genetic variation based on a model of isolation by distance.  We also introduce a simple statistic for quantifying a population's admixture status, and demonstrate the utility of this approach with two high profile empirical applications.

%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Methods}
Throughout these Methods, we will introduce example scenarios, simulated under a neutral, spatially structured coalescent process.  We will also present the results of SpaceMix analyses run on those data, both to illustrate its utility and to clarify our presentation.  For all details of simulation procedure and analyses, please see Appendix X.

\subsection*{Data}
Our data consist of $L$ unlinked variable loci sampled across $K$ populations, as well as the geographic coordinates, denoted $G_k$, (latitude and longitude) at which those populations were sampled (although $G_k$ may be missing for some or all of the populations.  For convenience, we use bi-allelic single nucleotide polymorphisms (SNPs) as the genetic data in our descriptions below, but we note that this method could be applied equally well to other types of data, including microsatellites.  We refer to the $K$ accessions as populations, but the method can also be applied to individuals.  We summarize these genetic data as a set of allelic count data and sample size data, arbitrarily choosing an allele to count at each locus, and denoting the number of the counted allele at locus $\ell$ in population $k$ as $C_{\ell,k}$ out of a total sample size of $S_{\ell,k}$ alleles.  The sample frequency of the counted allele at locus $\ell$ in population $k$ is therefore $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  Our method models the covariance in these allele frequencies between populations across loci.


%%%%%%%%% %%%%%%%%% 
\subsection*{The Normal Approximation to Drift}
In order to model this covariance, we draw on previous work (Cavalli-Sforza \& Edwards, Nicholson et al 2002, Coop et al 2010, Pickrell \& Pritchard 2012), and model the process of genetic drift, \gc{i.e. the compouding of binomial sampling over the generations,}  as approximately Gaussian. Briefly, if the population frequency of allele $\epsilon$ in an ancestral population, $A$ is $\epsilon_A$, then we model the population frequency of that allele in a daughter population,  $B$, as follows:
\begin{equation}
\label{eq:normal_drift}
\epsilon_B \sim N(\epsilon_A,\delta_B(\epsilon_A)(1-\epsilon_A))
\end{equation}
where the parameter  $\delta_B$ is the amount of drift separating $A$ and $B$, a parameter shared by all neutral loci in the entire genome.  The binomial variance term in the normal variance in \eqref{eq:normal_drift} ignores the fact that the drift variance will change from generation to generation due to changes in the frequency of $\epsilon$ within a population over time.  This approximation will work best for alleles at intermediate frequency, and in situations where the extent of drift is sufficiently limited (short time-scales or large population sizes). 

Among populations, the differentiating process of drift is counteracted by the homogenizing force of migration, so that populations with higher levels of historical or ongoing migration can be thought of as having more shared drift, i.e. stronger covariance in their allele frequency deviation around the ancestral (or global) mean. We model the population frequencies at a locus as multivariate normal (MVN) with mean $\epsilon$ and covariance matrix $\epsilon (1-\epsilon)\Omega$. The multivariate normal distribution offers a natural statistical framework for describing this covariance, which may be straightforwardly modeled as a parametric function of any pairwise distance variable (see, e.g., Bradburd, Ralph, and Coop 2013). 

% Unlike multivariate normal observations however, allele frequencies at variable loci are constrained to have a mean between 0 and 1, and may have heterogeneous variance across loci.  To more closely meet the assumptions of our statistical inference model, we mean-center and normalize our observations by the following procedure.

However, we do not have the population frequencies nor the `ancestral' frequency $\epsilon$. Instead we mean-center and normalize our observations at a locus using the weighted mean sample frequency in place of $\epsilon$.  Recall that the sample allele frequency at locus $\ell$ in population $k$ is given by $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  We wish to calculate a sample mean frequency at each locus weighted by the sample size in each population.  As sample size may vary across loci, we first calculate $\bar{S}_k$, the mean population sample size in population $k$, as $\bar{S}_k = \frac{1}{L}\sum_L S_{\ell,k}$.  We then calculate the weighted sample mean frequency at locus $\ell$ as follows:
\begin{equation}
\label{eq:sample_mean_freq}
\bar{f}_{\ell} = \frac{1}{\sum_K S_{\ell,k}} \sum_K \hat{f}_{\ell,k} S_{\ell,k}
\end{equation}
We approximate the binomial variance at each locus by $\bar{f}_{\ell}(1-\bar{f}_{\ell})$.  To avoid modeling the heterogeneous effect of this variance across loci, we standardize by this variance.  We call the standardized allele frequencies $X_{\ell,k}$, and calculate them as follows:
\begin{equation}
\label{eq:MCN_freqs}
X_{\ell,k} = \frac{ \hat{f}_{\ell,k} - \bar{f}_{\ell} } {\bar{f}_{\ell}(1-\bar{f}_{\ell})}
\end{equation}
Note, by using the sample mean frequency to mean-center our observations, we lose a degree of freedom, and reduce the covariance across loci between populations (sometimes inducing negative covariance among distant populations). We accomodate the extra sampling noise distortion, and reduced rank of the covariance matrix by assuming that our
\begin{equation}
X_{\ell} \sim MVN(0, \Omega^{\prime} )
\end{equation}
where $\Omega^{\prime}$ is a simple transform of $\Omega$. We discuss this further in Appendix A.

\gc{Okay so I think the right strategy re. the mean centering and dropping is treat everything in the main text like we've got the global freq. and then just apply that twist in the appendix}

%Note also that this approximation works best for alleles at intermediate frequency, which are less likely to be lost or fixed during the interval separating $A$ and $B$.  However, if the extent of drift sufficiently limited (time scales are short or population sizes are large), the approximation of inter-generational binomial sampling to Brownian motion is good

%The parameter $\delta_B$, the amount of drift that has occurred in $B$ since it split off from $A$, is approximately equal to $\frac{t}{2N_e}$, where $t$ is time in generations, and $N_e$ is the effective population size.  Note that the binomial variance term in the normal variance in \eqref{eq:normal_drift} ignores the fact that the drift variance will change from generation to generation due to changes in the frequency of $\epsilon$ within a population over time.  Note also that this approximation works best for alleles at intermediate frequency, which are less likely to be lost or fixed during the interval separating $A$ and $B$.  However, if the extent of drift sufficiently limited (time scales are short or population sizes are large), the approximation of inter-generational binomial sampling to Brownian motion is good (\gb{see figure comparing fit of binomial sampling to fit of normal approx?}).  

% Between populations, the differentiating process of drift is counteracted by the homogenizing force of migration, so that populations with higher levels of historical or ongoing migration can be thought of as having more shared drift.  Across loci, populations with highly shared drift will tend to covary more strongly in the deviations their allele frequencies take from some ancestral (or global) mean.

% %%%%%%%%% %%%%%%%%% 
% \subsection*{Mean-centering and normalizing variance}
% The multivariate normal distribution offers a natural statistical framework for describing this covariance, which may be straightforwardly modeled as a parametric function of any pairwise variable (see, e.g., Bradburd, Ralph, and Coop 2013).  Unlike multivariate normal observations however, allele frequencies at variable loci are constrained to have a mean between 0 and 1, and may have heterogeneous variance across loci.  To more closely meet the assumptions of our statistical inference model, we mean-center and normalize our observations by the following procedure.

% Recall that the sample allele frequency at locus $\ell$ in population $k$ is given by $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  We wish to calculate a sample mean frequency at each locus weighted by the sample size in each population.  As sample size may vary across loci, we first calculate $\bar{S}_k$, the mean population sample size in population $k$, as $\bar{S}_k = \frac{1}{\ell}\sum_\ell S_{\ell,k}$.  We then calculate the weighted sample mean frequency at locus $\ell$ as follows:
% \begin{equation}
% \label{eq:sample_mean_freq}
% \bar{f}_{\ell} = \frac{1}{\sum_k S_{\ell,k}} \sum_k \hat{f}_{\ell,k} S_{\ell,k}
% \end{equation}
% The binomial variance at each locus varies as a function of the global allele frequency, and, at locus $\ell$, is given by $\bar{f}_{\ell}(1-\bar{f}_{\ell})$.  To avoid modeling the heterogeneous effect of this variance across loci, we standardize by the binomial variance.  We call the mean-centered normalized allele frequencies $X_{\ell,k}$, and calculate them as follows:
% \begin{equation}
% \label{eq:MCN_freqs}
% X_{\ell,k} = \frac{ \hat{f}_{\ell,k} - \bar{f}_{\ell} } {\bar{f}_{\ell}(1-\bar{f}_{\ell})}
% \end{equation}
% We then model these transformed data as multivariate normal, as described in the next section. 



%%%%%%%%% %%%%%%%%% 
\paragraph{Spatial Covariance Model}
We wish to model the covariance between populations as the result of a spatial process, in which migration rates between nearby populations are higher than between distant ones, so that a population has higher covariance with a close neighbor than with a more distant population.  We choose as the form of our parametric covariance matrix a simple and flexible model, with an exponential decay of allele frequency covariance with geographic distance (Wasser et al 2004, Bradburd, Ralph, and Coop 2013).  The covariance between populations $i$ and $j$ is proportional to 
\begin{equation}
\label{eq:spatial_covariance}
\Omega_{i,j} = \frac{1}{\alpha_0} \text{exp} \left(	\left( \alpha_1D_{i,j} \right)^{\alpha_2} \right) + I\bar{S_k}^{-1} + I\eta_k \text{,}
\end{equation}
where $D_{i,j}$ is the geographic distance between population $i$ and $j$ (and therefore a function of their locations, $G_i$ and $G_j$), $\alpha_0$ controls the within-population variance, or the covariance when distance between points is 0 (the sill of the covariance matrix),  $\alpha_1$ controls the rate of the decay of covariance per unit pairwise distance, and $\alpha_2$ determines the shape of that decay.  In addition, we introduce population-specific variance terms to accommodate unshared drift and sample size effects.  These are given by $I\bar{S_k}^{-1} + I\eta_k$, where $I$ is the identity matrix, $I\bar{S_k}$ is the mean sample size in population $k$ across all loci, and $\eta_k$ is the nugget estimated in population $k$.  

This matrix $\Omega$ can be transformed into $\Omega^{\prime}$, the standardized sample frequency covariance matrix.
%\gc{what about the nugget?} \gb{it's complicated and a little confusing, because, as incorporated into the admixed covariance below, this covariance function doesn't get a nugget (that would be double-dipping the nugget).  So I'm not sure the best way to introduce the nugget.  Clearly, it ought to be here for the non-admixed model, so maybe we should introduce yet \emph{another} variable name, which would be the spatial covariance \emph{sans} nugget, and then use that in the admixed form of $\Omega$ below?}


Given the assumption of multivariate normality for our sample frequencies, it follows that the sample covariance of our standardized sample frequencies calculated across loci ($\widehat{\Omega} = X X^T$)  is Wishart distributed with degrees of freedom equal to the number of loci ($L$) across which the covariance is calculated.
That is, 
\begin{equation}
\label{eq:wishart_dist}
\widehat{\Omega} \sim \mathcal{W}\left( L^{-1} \Omega^{\prime}, L	\right)
\end{equation}
\gc{should the $L^{-1}$ be there? Need to work a bit to get this right in terms of transform, but I think this is the right way to go.} So we can estimate the parameters of our simple isolation by distance model, $\alpha_0,~\cdots, \alpha_2$, by treating \ref{eq:wishart_dist} as the likelihood of our standardized sample frequencies across loci, as this contains all of the information about our parameters of interest. \gc{Mention prior on params make this posterior?} Handily it also means that once the sample covariance matrix has been calculated all other computations do not scale linearly with number of loci, making the method scalable to genome size datasets. \gc{Have some link to an appendix describing what to do with linked loci?}


\gc{MOVE to appendix, or delete.
The likelihood of the data $X_{\ell,k}$, given the observed distance matrix $D$ between all $K$ populations and the values of the three $\alpha$ parameters ($\vec{\alpha}$), which together parameterize the spatial covariance matrix $\omega$, is given by
\begin{equation}
\label{eq:wishart_lnl}
\mathcal{P}(X_{\ell,k} \; | \; D(G), \pmb{\vec{\alpha}}) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\omega}^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\pmb{\omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}
where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface variables are parameters estimated as part of our inference procedure.
}


%%%%%%%%% %%%%%%%%% 
\paragraph{Accommodating non-equilibrium processes}

This model assumes that the variance in allele frequencies is the same in all locations and that the covariance between pairs of populations decays in the same way with geographic distance in all portions of the sampled range (i.e. that the process is homogeneous and isotropic). However, in many cases those assumptions will not be met. Non-equilibrium processes like long distance admixture, colonization, or population expansion events will distort the relationship between covariance and distance across the range.  Barriers to dispersal on the landscape can also change the relationship between allele frequency covariance and distance.

%bias the estimation of the contribution of a unit distance to decay in covariance.

%\gc{got rid of bias parameter estimates bits, as it's not clear what these parameters are anyway, esp. if model is very wrong}
%\gc{Or could intro. nugget here, as part of non-equilibrium}

One way that we can accommodate these heterogeneous processes is to allow populations to choose their own locations with respect to each other to maximize the resemblance to isolation by distance. Two populations that are sampled at distant locations but that are genetically similar (perhaps one was recently founded by a colonization event from the other) may choose estimated locations that are nearby, while two populations that are sampled close together, but that are genetically dissimilar (e.g, are separated by a barrier), may choose locations that are farther apart. The result is a \gc{``geogenetic''} map in which the distances between populations are indicative of the way that populations perceive the distances between themselves.  

To do this we can treat population's geographic locations as a random variable, and estimate them as part of our bayesian inference procedure.  We write our posterior as 
\begin{equation}
P(\pmb{G'}, \pmb{\vec{\alpha}}| X ) \propto  P(\widehat{\Omega} |\Omega (\vec{\alpha},\pmb{D}(\pmb{G'}) ) P(\pmb{\vec{\alpha}}) P(\pmb{G'})  
\end{equation}
where the constant of proportionality is the normalization constant of the posterior. The probability of the sample covariance matrix, $P(\widehat{\Omega} |\Omega (\vec{\alpha}\pmb{D}(\pmb{G'}) )$, is as before the Wishart likelihood of 
the sample covariance matrix of our standardized frequencies, and our priors on $\pmb{\vec{\alpha}}$ are as before. Here $P(\pmb{G'}) $ are the priors for the spatial locations, which we assume are independent across locations. For the prior on population $k$'s estimated location, $G'_k$, we use a bivariate normal spatial prior centered on the observed location $G_k$. We perform MCMC on all of the random variables; for more details on our Bayesian inference procedure see Appendix B. 
 
The sampled locations are a natural prior on where the populations are positioned under the null of isolation by distance (as well as a natural starting position for the MCMC). This prior also encourages the resulting inferred geogenetic map to be anchored in the observed locations and to represent (informally) the minimum distortion to space necessary to satisfy constraints placed by genetic similarities of populations. However, one concern is that if the method returns a spatial configuration that strongly resembles a map, this could reflect the possibility that there is insufficient information in the data to overcome the influence of the prior, or that the MCMC having not been run long enough.  To address this concern we can also use random locations as the ``observed locations" ($G_k$) to remove any influence of the observed map on the output via the prior, or we can change the variance on the spatial priors to ascertain the effect of the prior on inference.

%remove ir more to approxim
% Our likelihood function is now given by,
% \begin{equation}
% \label{eq:choose_location_wishart_lnl}
% \mathcal{P}(X_{\ell,k} \; | \; \pmb{D}(\pmb{G'}), \pmb{\vec{\alpha}}) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\omega}^{-1}Y)}{2}	\right) }	
% 									{2^{\frac{LK}{2}}  |\frac{\pmb{\omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
% \end{equation}
% where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface variables are parameters estimated as part of our inference procedure.  Notice that this equation has the same form as \eqref{eq:wishart_lnl}, save that the pairwise geographic distance matrix $D$ is now a parameter to be estimated rather than an observed value.

To illustrate this inference procedure, we present several scenarios simulated under the coalescent (using \textrm{ms}, (Hudson)), as well as the output of SpaceMix analyses run on them.  In all scenarios, we simulate variations of a stepping stone model, with populations  arranged on a grid and symmetric nearest neighbor migration (\ref{sfig:simple_lattice}), logging data from every other population.

In the first scenario (output illustrated in Fig. \ref{sfig:lattice_scenarios}\subref{lattice_inference}), we simulate a stepping stone model at migration-drift equilibrium and homogeneous migration rates across the grid.  In our second scenario (Fig. \ref{sfig:lattice_scenarios}\subref{barrier_inference}), we simulate under the same stepping stone model, but introduce a longitudinal barrier to dispersal (between populations 11:15 and 16:20, at \textit{x} = 6.01), across which migration rates are attenuated by a factor of 5.  In the third scenario (Fig. \ref{sfig:lattice_scenarios}\subref{expansion_inference}), we simulate an expansion event, in which, all populations in the last five columns of the grid have expanded in the recent past from the nearest population in their row (referring to Fig. \ref{sfig:simple_lattice}, populations 25 and 30, as well as the three unsampled populations that bracket them, have all expanded at the same point in the recent past from population 20).

On all simulated datasets, we perform SpaceMix analyses in which we treat population locations as random variables to be estimated as part of the model, and randomly place the `observed' location of each population so as to remove the influence of the prior.  For clarity, we present the full Procrustes superimposition of the inferred locations around the coordinates used to simulate the data.

%% Perhaps just say this in the methods section (attenuated by $\sqrt{2}$ on the diagonals) and include file of ms command lines. 
\begin{figure}
	\centering
		{\includegraphics[width=2.4in,height=2in]{figs/stationary_pops_map.png}}
	\caption{Simulation scenario: populations on a lattice with symmetric nearest-neighbor migration, sampling every second population.}
\label{sfig:simple_lattice}
\end{figure}

\begin{figure}
	\centering
		\subcaptionbox{Lattice \label{lattice_inference}}
			{\includegraphics[width=1.85in,height=1.54in]{figs/GeoGenMap_lattice.pdf}}
		\subcaptionbox{Barrier \label{barrier_inference}}
			{\includegraphics[width=1.85in,height=1.54in]{figs/GeoGenMap_barrier.pdf}}
		\subcaptionbox{Expansion  \label{expansion_inference}}
			{\includegraphics[width=1.85in,height=1.54in]{figs/GeoGenMap_expansion.pdf}}
	\caption{Population maps inferred using SpaceMix under three different scenarios: a) simple lattice at equilibrium; b) a lattice with a barrier across the center line of longitude; c) a lattice with recent expansion on the eastern margin.}\label{sfig:lattice_scenarios}
\end{figure}

In Figure \ref{sfig:lattice_scenarios}\subref{lattice_inference}, the reader can see that the configuration estimated for the populations by SpaceMix matches the lattice structure used to simulate the data, and that populations are correctly choosing their nearest neighbors.  Similarly, in Figure \ref{sfig:lattice_scenarios}\subref{barrier_inference}, the population configuration matches that of the lattice used to simulate the data, but due to the influence of the barrier, the two halves of the map have pushed farther away from one another.  This gap between populations on either side of the barrier reflects the way those populations perceive the increased effective distance between them.  In the scenario of recent expansion (Fig. \ref{sfig:lattice_scenarios}\subref{expansion_inference}), the daughter populations of the expansion event cluster with their parent populations, reflecting the higher relatedness (per unit geographic separation) between them.

\gc{Perhaps give the covariance vs distance plots in supplement.}

If our data are well fit by a model of isolation by distance then (1) a population's genetic makeup should be well predicted by that of its neighbors, and (2) populations should not show excess covariance with distant populations. Violation of either of these two points will result in a poor fit of a simple isolation by distance model, and particularly a violation of point (2) may indicate that long distance admixture has taken place.  

To examine the behavior of SpaceMix when there is long distance covariance between populations, we simulated an admixture event on the stepping-stone model we had used previously.  Specifically, (using Fig. \ref{sfig:simple_lattice} as a reference) we allowed population 30, in the northeast corner of the grid, to draw half of its ancestry from population 1, in the southwest corner.  The result of a SpaceMix analysis in which the locations of these populations were estimated is shown in Figure \ref{sfig:admixture_inference_CYOL}.

\begin{figure}
	\centering
	\includegraphics[width=2.4in,height=2in]{figs/GeoGenMap_corner_admixture_CYOL.png}
	\caption{Inference of population locations in the scenario depicted in Figure \ref{sfig:admixture_scenario}.  Population 30 has received half of its lineages from population 1, to simulate a long distance admixture event in the very recent past.}\label{sfig:admixture_inference_CYOL}
\end{figure}

This signal of excess covariance over anomalously long distances is clearly difficult to accommodate within the ``choose-your-own-location" framework described above.  In Figure \ref{sfig:admixture_inference_CYOL}, the reader can see the torturous lengths to which the method goes to come up with a configuration of populations that accommodates their genetic relationships.  The admixed population 30 is estimated to have a location intermediate between population 1, the source of its admixture, and populations 24, 25, and 29, the nearest neighbors to the location of its non-admixed lineages.  However, this warping of space is difficult to interpret, especially in the visualization of genetic relationships in empirical data for which a researcher does not know the true demographic history.  It would therefore be of great utility to directly model the action of admixture on spatial patterns of genetic variation.

%%%%%%%%% %%%%%%%%% 
\subsection*{Inference of Spatial Admixture}

We can incorporate recent admixture directly into out inference framework.  We imagine that population $k$ draws the majority of its ancestry from $G_k$, but a proportion $p_k$ of its ancestry comes from another location \kadmixsource{k}, which we refer to as its source of admixture. The mean standardized population allele frequency at locus $\ell$ in population $k$ is a weighted average of the allele frequencies at the geographic location of the sampled population and those at the coordinates of the source from which the observed population draws admixture:
\begin{equation}
p_k f_{\ell,k} + (1-p)f_{\ell,k*} \label{eqn-admixedfreq}
\end{equation}
where $f_{\ell,k}$ are the model-estimated allele frequencies at locus $\ell$ at the spatial location of population $G_k$ and $f_{\ell,k*}$ are the model-estimated allele frequencies at the spatial location of the source of admixture \kadmixsource{k}. We can allow each of our populations to have this setup, each with an independent spatial source of admixture. 


Following from the form  of eqn. \eqref{eqn-admixedfreq} the covariance between the standardized allele frequencies of population $i$ and $j$ can be modeled as 
\begin{alignat}{3}
\label{eq:admixed_covariance_1}
\Omega_{i,j}^{(A)} = (1-p_i)(1-p_j) \Omega_{i\;,\;j\;} \; \times&\\
(p_i)(1-p_j) \Omega_{\identifyadmixsource{i},\;j\;} \; \times   \notag&\\
(p_j)(1-p_i) \Omega_{i\;,\;\identifyadmixsource{j}} \; \times   \notag&\\
(p_i)(p_j) \Omega_{\identifyadmixsource{i},\;\identifyadmixsource{j}} \; +   \notag&\\
\delta_{i,j} \eta_i \notag&
\end{alignat}
where $\identifyadmixsource{i}$ and $\identifyadmixsource{j}$ are the sources from which populations $i$ and $j$ are drawing their admixture with proportions $p_i$ and $p_j$. $\Omega$ is the spatial covariance function parameterized by the pairwise geographic distances between each pair of populations, \emph{without} the population-specific variance terms on the diagonal (i.e., in Eqn. \ref{eq:admixed_covariance_1}, i.e. $\Omega_{i,j} = \frac{1}{\alpha_0} \text{exp} \left(	\left( \alpha_1D_{i,j} \right)^{\alpha_2} \right)$). Note that we then reintroduce the nugget for each population $\eta_i$, to model drift or excess variance in population $i$ on top of that predicted by our mixture of frequencies predicted by our spatial model.
  The admixed covariance between populations $i$ and $j$, $\Omega_{i,j}^{(A)}$ is then a function of all the pairwise spatial covariances between populations $i$ and $j$ and the points from which they draw admixture, $\identifyadmixsource{i}$ and $\identifyadmixsource{j}$.  Those spatial covariances in turn are a function of all combinations of pairwise distances between their locations: $G_i$, $G_i$, \kadmixsource{i}, and \kadmixsource{j}.  This parametric covariance form is illustrated in Figure \ref{sfig:admixed_cov_diagram}.

\gc{ $\delta_{i,j} $ is the indicator function $1$ if $i=j$ $0$ otherwise. }


%where $f_{\ell,i}$ is the allele frequency in population $i$, $f_{\ell,j}$ is the allele frequency in population $j$, and $p$ is the admixture proportion, which varies between 0 and 1 and describes the extent to which populations $i$ and $j$ are contributing to the genetic make-up of population $k$.

% To infer the spatial context of this admixture, we allow each population a point in space, which we refer to as its source of admixture, from which it draws its admixture, and we model both the location of that source and the extent (proportion) of that admixture.  The observed allele frequencies in sampled populations are therefore a weighted average of the model-estimated allele frequencies at the geographic location of the sampled population and those at the coordinates of the source from which the observed population draws admixture.  That is, the observed allele frequencies in population $k$ are modeled as follows:
% \begin{equation}
% f_{k} = pf_{k'} + (1-p)f_{j},
% \end{equation}
% where $f_{k'}$ are the model-estimated allele frequencies across loci at the spatial location of population $k$ and $f_{j}$ are the model-estimated allele frequencies at the spatial location of the source of admixture $j$, from which population $k$ is drawing admixture in proportion $p$.  The admixture proportion $p$ is constrained to vary between 0 and 0.5, such that at least half of a population's genetic make-up must be determined by its geographic location.


% We re-introduce the population-specific variance terms on each diagonal element of this admixed covariance matrix.  The full expression for our admixed covariance function is below.
% \begin{alignat}{3}
% \label{eq:admixed_covariance_2}
% \Omega_{i,j} = (1-p_i)(1-p_j) \omega_{i\;,\;j\;} \; \times&\\
% (p_i)(1-p_j) \omega_{i',\;j\;} \; \times   \notag&\\
% (p_j)(1-p_i) \omega_{i\;,\;j'} \; \times   \notag&\\
% (p_i)(p_j) \omega_{i',\;j'} \; +   \notag&\\
% \delta_{i,j} \bar{S_k}^{-1} + \delta_{i,j} \eta_k \phantom{+} \notag&
% \end{alignat}

% where $I$ is the identity matrix, $\bar{S_k}$ is the mean sample size in population $k$ across all loci, and $\eta_k$ is the nugget estimated in population $k$.


As we only get to observe the sample frequencies and we standardize our allele frequencies using the sample mean our predicted admixture covariance matrix needs to be transformed to accommodate these sampling considerations. We can do this as before see methods XXXX. We again treat the likelihood of our sample covariance matrix as Wishart now given the parametric covariance matrix $\Omega^{(A)}$ as specified by the parameters $\vec{p}$, \admixsource, $G$,$\vec{\alpha}$, and $\vec{\eta}$. We treat the location of the source of admixture for population $k$, \kadmixsource{k}, and the population's admixture proportion, $p_k$, as random variables and jointly estimated as part of our inference procedure. \gc{state form of priors.}


\begin{figure}[ht!]
	\centering
	\includegraphics[width=2in,height=2in]{figs/admix_cov_fig.pdf}
	\caption{An illustration of the form of the admixed covariance given in Eqns. \ref{eq:admixed_covariance_1} and \ref{eq:admixed_covariance_2}.  Populations $i$ and $j$ are drawing admixture in proportions $p_i$ and $p_j$ from their respective sources of admixture, $i'$ and $j'$, and all pairwise spatial covariances (the $\omega$'s) are shown.  In this cartoon example, population $j$ is drawing more admixture from its source $j'$ than $i$ is from its source $i'$ (i.e., $p_j > p_i$).}\label{sfig:admixed_cov_diagram}
\end{figure}


% The likelihood function for this model in which each population is allowed to draw admixture from a point in space is 
% \begin{equation}
% \label{eq:choose_admixture_wishart_lnl}
% \mathcal{P}\left(X_{\ell,k} \; | \; \pmb{D}\left(G,\pmb{G_{'}}\right), \pmb{\vec{\alpha}}, \pmb{p}, S, \pmb{\eta}\right) = \frac{|Y|%^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\Omega}^{-1}Y)}{2}	\right) }	
% 									{2^{\frac{LK}{2}}  |\frac{\pmb{\Omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}%{2} \right)	},
% \end{equation}
% where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface %variables are parameters estimated as part of our inference procedure.

Using the example admixture scenario described above and used in the analysis depicted in Figure \ref{sfig:admixture_inference_CYOL}, we demonstrate the inference of populations' sources and strengths of admixture and illustrate the results in Figure \ref{sfig:corner_admixture_just_adinf}.  The reader can see that only the admixed population (population 30) is drawing admixture from the location of the source of admixture that was used to simulate the data, and that all other populations, which are not admixed, are choosing to draw admixture in only negligible amounts.

\begin{figure}[ht!]
	\centering
	\includegraphics[width=2.4in,height=2in]{figs/GeoGenMap_corner_admixture_adinf.png}
	\caption{Posterior distribution of inference of the sources and strengths of admixture for the sampled populations.  The admixed population (Population 30) is drawing admixture from the location of its source of admixture that was used to simulate the data (the location of Population 1).}\label{sfig:corner_admixture_just_adinf}
\end{figure}

\subsection*{Models}
The models described above may be used in various combinations.  In the simplest model, populations may not choose their own locations, nor are they allowed to draw admixture, and the only parameters to be estimated are those of the spatial covariance function given in Eqn \eqref{eq:spatial_covariance}, and the population-specific variance terms ($\eta_k$).  In the most complex model, population locations, the locations of their sources of admixture, and the proportions of that admixture are all estimated jointly in addition to the parameters of the spatial covariance function and the population specific variances.  The full likelihood of this most complex parameterization is given by,
\begin{equation}
\label{eq:source_and_target_wishart_lnl}
\mathcal{P}\left(X_{\ell,k} \; | \; \pmb{D}\left(\pmb{G'},\pmb{G_{'}}\right), \pmb{\vec{\alpha}}, \pmb{p}, S, \pmb{\eta}\right) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\Omega}^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\pmb{\Omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}

\gb{A note here about how this complex model is still identifiable}

To demonstrate the use of the model in which the location of each population as well as the location of its source of admixture are estimated jointly, we used the spatial stepping-stone coalescent simulation procedure described above to generate a dataset of populations on a lattice in which there is both a barrier to dispersal and a more subtle admixture event (admixture proportion = 10\%, see Fig. \ref{sfig:barr_inland_ad}\subref{barr_inland_ad_scenario}).  In the SpaceMix analysis (Fig. \ref{sfig:barr_inland_ad}\subref{barr_inland_ad_inference}), the separation of the east and west sides of the grid accommodates the effect of the barrier to migration, and the admixed population (population 23) chooses admixture from very close to its true source (population 13), and in close to the correct amount ($\bar{p} = 0.05; 95\% \text{ credible interval} = 0.02-0.08$). \gb{discuss how prior forces amount down, so it's cheaper to choose more westward pop as source}.

\begin{figure}[ht!]
	\centering
		\subcaptionbox{Lattice \label{barr_inland_ad_scenario}}
			{\includegraphics[width=2in,height=1.66in]{figs/bar_inland_ad_scenario.pdf}}
		\subcaptionbox{Barrier \label{barr_inland_ad_inference}}
			{\includegraphics[width=2in,height=1.66in]{figs/GeoGenMap_barr_inland_admixture_1.png}}
	\caption{Population maps inferred using SpaceMix under three different scenarios: a) simple lattice at equilibrium; b) a lattice with a barrier across the center line of longitude; c) a lattice with recent expansion on the eastern margin.}\label{sfig:barr_inland_ad}
\end{figure}
%This most complex model is, in many cases, nonetheless identifiable.  Although in certain scenarios, a population may choose a location nearer to its source of admixture, rather than 
\newpage
\section*{Inference}
For details on our Bayesian inference framework and Markov chain Monte Carlo inference procedure, please see the Section: How I spent the past year!

\section*{Empirical Applications}
To demonstrate the applications of this novel method, we employed it in two canonical empirical systems: the greenish warbler ring species complex, and a global sampling of contemporary human populations.

\subsection*{Greenish Warblers}  %a canonical proposed examples of a ring species (Irwin et al 2001).  The complex 
The greenish warbler (\emph{Phylloscopus trochiloides}) species complex is broadly distributed around the Tibetan plateau, and exhibits gradients around the ring in a range of phenotypes including song, as well allele frequencies (Ticehurst (1938),Irwin et al 2001, Irwin et al 2005, Irwin et al 2008).  At the northern end of the ring in central Siberia, where the eastern and western arms of population expansion meet, there are discontinuities in call and morphology, as well as a genetic discontinuity and reproductive isolation (Irwin et al 2001, Irwin et al 2008). It is proposed that the species complex represents a ring species, in which selection and/or drift, acting in the populations as they spread northward on either side of the Tibetan plateau, have led to the evolution of reproductive isolation (REFs).  The question of whether it constitutes a ring species, in purest form, focuses on whether gene flow along the margins of the plateau has truly been continuous throughout the history of the expansion or if, alternatively, discontinuities in migration around the species complex's range have facilitated periods of differentiation in genotype or phenotype without gene flow (Mayr 1942, Mayr 1970, Coyne and Orr 2004).  However, we note that many would still classify this as a ring species even if that condition were not met, just not as a case of speciation-by-distance \citep[see][ for discussion]{}.


\gc{Based on a larger SNP dataset Alcaide et al (2014),} have suggested that the greenish warbler species complex constitutes a `broken' ring species, in which there have been historical discontinuities in gene flow that facilitated the evolution of reproductive isolation between adjacent forms.  Because the questions in this system are fundamentally both geographic and genetic in nature, it is eminently SpaceMix-able, and, within this spatial framework, we performed a number of analyses to investigate the geographic context of population differentiation in the greenish warbler species complex. For these analyses, we used the dataset from Alcaide et al (2014), which consisted of 95 individuals sampled at 22 distinct locations and sequenced at 2,334 SNPs, of which 2,247 were bi-allelic and retained for SpaceMix runs. 

% The analysis procedure is detailed in Appendix XXX.  We ran two analyses using the observed population locations as the prior on $G'$.  Then, to assess the potential influence of the spatial prior on population locations, we ran one analysis in which random, uniformly distributed locations between, for longitude, the minimum and maximum observed longitude, and, for latitude, the minimum and maximum observed latitude were used as the prior on population locations.  We then repeated these analyses, but treated each sequenced individual as its own population.  For clarity and ease of interpretation, we present a full Procrustes superimposition of the inferred population locations ($G'$) and their sources of admixture (\admixsource{}), using the observed latitude and longitude of the populations/individuals ($G$) to give a reference position and orientation.  As results were generally consistent across multiple runs for each dataset regardless of the prior employed we (unless stated otherwise) present only the results from the `random' prior analyses.


% http://www.jstor.org/discover/10.2307/3893431?uid=3739560&uid=2129&uid=2&uid=70&uid=4&uid=3739256&sid=21104323343631

%At stake is whether the species offers an example of sympatric speciation (i.e. speciation with continuous gene flow).


\begin{figure}
	\centering
		\subcaptionbox{Warbler subspecies distribution map \label{irwin_map}}
			{\includegraphics[width=2.4in,height=2in]{figs/Irwin_warbler_map_figure.png}}
		\subcaptionbox{Inferred admixture proportions and population nuggets \label{warb_pop_adnug}}			
			{\includegraphics[width=2.4in,height=2in]{figs/population_warbler_admix_values_nugget.png}}
		\subcaptionbox{Map inferred using SpaceMix \label{warb_pop_no_arrows}}
			{\includegraphics[width=2.4in,height=2in]{figs/population_warbler_map_no_arrows.png}}
		\subcaptionbox{Map inferred using SpaceMix with admixture arrows shown \label{warb_pop_arrows}}
			{\includegraphics[width=2.4in,height=2in]{figs/population_warbler_map_randpr1.png}}
	\caption{Greenish warbler subspecies distributions contrasted with maps of all 22 sampled populations inferred using SpaceMix.  For clarity of presentation, the inferred coordinates and parameter values are taken from the single draw of the MCMC with the highest posterior probability and have have been Procrustes transformed around the coordinates of the lattice used to simulate the data. (a) Greenish warbler subspecies distribution map (from Irwin et al NNN).  (b) Inferred population admixture proportions and nugget parameters. (c) Inferred population map without admixture arrows shown, with population labels colored as in (a). (d) Inferred population map with admixture arrows (with thickness proportional to inferred admixture proportion) shown, with population labels colored as in (a). }\label{sfig:warbler_pops}
\end{figure}

\gc{run method w. no admixture. I think we should define the random priors in the methods above, as it's not specific to any of the analyses}

\gc{We first ran spacemix on the population dataset, with no admixture, setting the prior locations of the populations at random (as described above).} The inferred map (Figure \ref{sfig:warbler_pops}) largely recapitulates the geography of the sampled populations.  Populations choose locations around a large ring, with ordering similar to that of their true geographic locations.  The Turkish population (\textit{Phylloscopus trochiloides} ssp. \textit{nitidus}) clustered with the populations in the subspecies \textit{ludlowi}, but also chose a relatively high nugget parameter, reflecting the independent drift it does not share with its \textit{ludlowi} neighbors.  The Yekat population of \textit{viridanus} individuals clusters closely with the other, less far-flung \textit{viridanus} individuals, indicating that differentiation within that subspecies is not commensurate with the amount of IBD expected for samples separated by that much distance. 

In the north, where the twin waves of expansion around the Tibetan Plateau are hypothesized to meet, the inferred \gc{geogenetic} distance between populations identified as \textit{Phylloscopus trochiloides} ssp. \textit{plumbeitarsus} and ssp. \textit{viridanus} was \gc{much} greater than their observed geographic separation, reflecting the reproductive isolation between these adjacent forms \gc{{\bf POINT to graph showing this in supp}.  Interestingly, the ST population, which consists of six individuals sampled in Stolby, Russia, chooses a location intermediate between the \textit{plumbeitarsus} and \textit{viridanus} groups. The Stolby sample is composed of three individuals that belong to the eastern \textit{plumbeitarsus} and three individuals that belong to the western \textit{viridanus} \cite{Alcaide et al (2014)}. In the case where no admixture is allowed this population is forced to adopt an intermediate position to incorporate its admixed nature. }

%However, where the population-level analysis shows the Stolby population intermediate between the \textit{viridanus} and \textit{plumbeitarsus} clusters, the individual-level analysis reveals that the Stolby population (n=6)

We then ran the method allowing admixture, and again discuss the random priors results. The Stolby population chooses the highest admixture proportion, with a mean of 0.19 \bf{95\% credible intervals}.  Multiple runs agreed well on the level of admixture of the Stolby (see caption of Supplementary Figure \ref{sfig:warbler_pop_compare}). What does vary across runs, is whether the Stolby population chooses to locate itself by the \textit{viridanus} cluster and draw admixture from near the \textit{plumbeitarsus}  cluster or vise versa, however, this is to be expected given the 50/50 nature of the sample (Supplementary Figure \ref{sfig:warbler_pop_compare}). 

%\gc{Graham: dont feel like we need the three analyses in main text.} Across the three independent analyses (two with the observed population locations as spatial priors on the locations, $G'$, that they choose for themselves, one with random locations as spatial priors), the inferred values of admixture proportion are consistent (95\% credible intervals:  0.146-0.233, 0.154-0.242, 0.146-0.238).  However, both the position that the Stolby population chooses for itself and the region from which it draws admixture varies between runs (Figure \ref{sfig:warbler_pop_compare}).  In two of the analyses, the Stolby population moves to a position proximate to the \textit{viridanus} cluster and chooses admixture from a point beyond the \textit{plumbeitarsus} cluster, and in one analysis, this pattern is reversed.  

\begin{figure}
	\centering
		\subcaptionbox{\label{warb_pop_realpr1}}
			{\includegraphics[width=1.85in,height=1.54in]{figs/population_warbler_map_realpr1.png}}
		\subcaptionbox{\label{warb_pop_realpr2}}			
			{\includegraphics[width=1.85in,height=1.54in]{figs/population_warbler_map_realpr2.png}}
		\subcaptionbox{\label{warb_pop_randpr1}}
			{\includegraphics[width=1.85in,height=1.54in]{figs/population_warbler_map_randpr1.png}}
	\caption{Comparison of inferred maps from three independent analyses.  (a,b) Results from analysis using observed locations as priors on population locations.  (c) Results from analysis using random, uniformly distributed locations within the observed range of latitude and longitude as priors on population locations.}\label{sfig:warbler_pop_compare}
\end{figure}

Because \textit{a priori} assigned population membership may be artificial (individuals from more than one population may be sampled at a single site), we repeated these analyses on an individual level.  In these analyses, the sample size in each `population' was 2 (for the two alleles in a diploid), and each individual chose its own location as well as the location of its source of admixture, the proportion of that admixture, and its nugget.  As with the analysis on multi-sample populations, the results \gc{approximately} mirror the geography of the individuals.  \gc{Individuals choose very low levels of admixture...}

%The order of individuals' estimated locations around the Plateau is largely concordant with their true locations, and, in most cases, individuals sampled from the same population cluster closely together, as would be expected if samples collected at a single site are draws from a population that is locally panmictic.  

\begin{figure}
	\centering
		\subcaptionbox{Inferred map of warbler individuals \label{warb_ind_map}}
			{\includegraphics[width=2.8in,height=2.3in]{figs/individual_warbler_map_noarrows_randpr1.png}}
		\subcaptionbox{Closeup of non-\textit{nitidus} samples \label{warb_ind_map_closeup}}
			{\includegraphics[width=2.8in,height=2.3in]{figs/individual_warbler_map_noarrows_closeup_randpr1.png}}
	\caption{Inferred maps for individual warbler individuals, colored by subspecies. (a) complete map including Turkish \textit{nitidus} samples.  (b) close up of all non-\textit{nitidus} samples.}\label{sfig:warbler_inds}
\end{figure}

There are a number of obvious departures in the individual inferred geogenetic map from the observed map. The most obvious again is the clear split between \textit{viridanus} and \textit{plumbeitarsus} individuals in the north at the contact zone of the two waves of expansion.  This is clearer now than in the population-based analysis as individuals from the Stolby population have moved to near their respective \textit{viridanus} \textit{plumbeitarsus}  clusters. 

Despite the fact that \textit{viridanus} and \textit{plumbeitarsus} individuals have moved away from each other in our geogenetic map, they are still closer to each other than we might expect if their drift is truely independent (e.g. our populations could form along a line). This horseshoe, with \textit{viridanus} and \textit{plumbeitarsus} at its tips, is steady within and among runs of the MCMC and choice of position priors (see Supplementary clouds NNN). Is this biologically meaningful? A somewhat similar horseshoe shape appears when a principal components (PC) analysis is conducted and individuals are ploted on the first two PCs (Alcaide et al (2014), see our Supp. Figure NNN). However, as discussed by Novembre \& Stephens such patterns in PC analysis can arise for somewhat unintuitive reasons. If populations are simulated under a one dimensional stepping stone model, then plotting individuals on the first two PCs results in a horseshoe (e.g. see Supp. Figure NNN) not because of any particular gene flow connection between the tips but rather because of the othogonality requirement of PCs (see Novembre \& Stephens for more discussion). When spacemix is applied to one dimensional stepping stone data, the placement of samples is consistent with a line. In addition when we run Spacemix on the greenish warbler individuals specifying their location priors to fall along a straight line, with samples located at their approximate postions around the horseshoe, the posterior positons of the populations still curl up to form a horseshoe. The proximity of \textit{viridanus} and \textit{plumbeitarsus}  in geogenetic space may be due to gene flow between the tips of the horseshoe north of the Tibetan Plateau. This conclusion is in agreement with that of Alcaide et al (2014) who observed evidence of hybridization between \textit{viridanus} and \textit{plumbeitarsus} using assignment methods.

%\gc{One idea if we wanted to test this }

%that the individual-level analysis showed a

%Finally, we compared the SpaceMix map to a map derived from a Principal Components Analysis (Patterson and Reich 2006).  For this analysis, we calculated the eigendecomposition of the mean-centered allelic covariance matrix, then plotted individual's coordinates on the first two eigenvectors (e.g. Novembre et al 2008).  For clarity of presentation, we show the full Procrustes superimposition of the PC coordinate space around the geographic sampling locations of the warbler individuals (Figure NNN).  The concordance between the PC map and the SpaceMix map is generally quite good, and we discuss the interpretation of the geography implied by this map further below.
%\gb{or should I discuss it here and include the simulation figs showing how PC gets it right for the wrong reasons?}


%The removal of the Stolby population further highlights the inferred split, incommensurate with their observed geographic separation, between \textit{viridanus} and \textit{plumbeitarsus} in the North.

A second difference between the observed and inferred maps is a pair of individuals, one identified as \textit{P. t. ludlowi} (Lud-MN3), one as \textit{P. t. trochiloides} (Tro-LN11), that choose locations very close to one another and also away from the other individuals sampled at their locations. Examing pairwise sequence difference shows that these two individuals show unusually recent common ancestry (see SuppMat Figure NNN), and therefore are likely expressing their shared ancestry (drift unshared with other \textit{ludlowi} and \textit{trochiloides} individuals) by choosing locations that are close to each other and far from their respective clusters of individuals that were sampled at the same sites. \gc{move the commented out bit below to the caption of the pairwise seq. plot. }

% To investigate the potential reason for this behavior, we calculated average pairwise sequence divergence at the 2,247 polymorphic loci in the dataset between all 95 individuals and plotted it against the pairwise geographic distance between the individuals (see SuppMat Figure NNN).  The pairwise sequence divergence (0.103) at polymorphic loci between Lud-MN3 and Tro-LN11 is significantly lower than that between any other pair of individuals separated by a comparable distance - lower, in fact, than any comparison between individuals that were not co-located, and lower than any pairwise divergence between any pair of individuals save that between the two Turkish \textit{nitidus} samples. 

The SpaceMix map also diverges from the observed map in the distribution of individuals from the subspecies \textit{ludlowi}.  These samples were taken from seven sampling locations along the southwest margin of the Tibetan Plateau, but, in the SpaceMix analysis, partition into two main clusters, one near the \textit{trochiloides} cluster, and one near the \textit{viridanus} cluster.  This break between samples from the same subspecies, which is concordant with the findings of Alcaide et al (2014), makes the \textit{ludlowi} cluster unusual compared to the estimated spatial distributions of the other subspecies (see SuppMat Figure NNN).

\begin{figure}
	\centering
	\includegraphics[width=2.4in,height=2in]{figs/warb_ind_PC_map.png}
	\caption{The map of warbler individuals derived from a Principal Components analysis.}\label{warb_ind_PC_map}
\end{figure}

%

\subsection*{Human Populations}
Human population 
For our analysis of spatial patterns of human population structure, 

Description of the research questions we use SpaceMix to address

Details of analyses run

%%%%%%%%% %%%%%%%%% 
\section*{Results}
\gb{should we just merge the results section in with the empirical applications section?  not sure what else would go here.}

%%%%%%%%% %%%%%%%%% 
\section*{Discussion}
In this paper we have presented blah blah blah.  We believe this represents an advance over previous methods because blah blah blah.  This method can be used to answer a variety of empirical questions, including blah, blah, and blah, and also serves as an intuitive data visualization tool.

%%%%%%%%% %%%%%%%%% 
\subsection*{Empirical Results}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Greenish Warblers}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Humans}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\section*{Future Directions}

spatiotemporal model

spatialSTRUCTURE
%%%%%%%%%

\section*{Appendix}
The analysis procedure went as follows: 
\begin{itemize}
\item[1.] five independent chains were run for 5e6 generations each in which populations were allowed to choose their own locations (but no admixture).  Population locations were initiated at the origin (i.e. - at generation 1 of the MCMC, $G'_i = (0,0)$), and all other parameters were drawn randomly from their priors at the start of each chain.  
%
\item[2.]The chain with the highest posterior probability at the end of the analysis was selected and identified as the``Best Short Run".
%
\item[3.] A chain was initiated from the parameter values in the last generation of the Best Short Run.  Because inference of admixture proportion and location was not allowed in the five initial runs, admixture proportions were initiated at 0 and admixture locations, \admixsource{G} were initiated at the origin.  This  chain (the ``Long Run") was run for 1e8 generations, and sampled every 1e5 generations for a total of 1e3 draws from the posterior.
\end{itemize}




\end{document}













%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%	TEXT GRAVEYARD
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%% %%%%%%%%% 

 in which we estimate the 2-dimensional configuration of populations that, along with a model of the decay of covariance in allele frequencies with distance, best describes their empirical patterns of genetic differentiation
 
\subsection*{Spatial Admixture Statistic}

\gc{Do we also want to describe comparison of observed to expected covariance matrix?? Guess we could do it above, to show that our model has successfully improved the fit to the data.}

 
\gb{Figure 2: MS simulation of pops on a grid with 1 admixture event from a pop on one side to a pop on the other}

\gb{3a figure to show the pairwise f-stat between all pops and the admixed pop}

\gb{3b figure to show landscape of admixture to the admixed pop}

If our data are well fit by a model of isolation by distance then (1) a population's genetic makeup should be well predicted by that of its neighbors, and (2) populations should not show excess covariance with distant populations. Violation of either of these two points will result in a poor fit of a simple isolation by distance model, and particularly a violation of point (2) may indicate that long distance admixture has taken place.  We develop simple statistics, which we call spatial $f$-statistics \gb{should we give these a real name?}, to test these points \gc{do we actually test 1?}.

These statistics are in the spirit of both the $f_n$ statistics introduced by Reich et al (2009) and the admixture arrows in a TreeMix graph (Pickrell and Pritchard 2012), and are designed to pick up a signal of covariance between populations in excess of that predicted by our model.  These previous approaches use a null model of a branching population phylogeny and develop tests to diagnose covariance between populations greater than that predicted by the population tree.  Specifically, the tests identify violations of conditional independence of allele frequencies among populations that indicate that there has been admixture (gene flow) between disparate parts of the tree.  In place of a phylogeny, our tests use a spatial model of allelic covariance to look for population pairs whose sample covariance exceeds that predicted by their observed distance and the inferred parameters of the spatial model. %excess covariance between distant populations, indicative of excess gene flow.

%in which one population's covariance with another is expected to decay at a fixed manner across the landscape, as determined by the estimated parametric form of the spatial covariance.  

The details of this procedure are given below, and illustrated with a simulated example scenario (see Figure \ref{sfig:admixture_scenario}), featuring an admixture event.

\begin{figure}
	\centering
	\includegraphics[width=2.4in,height=2in]{figs/admixture_map.png}
	\caption{In this simulated scenario, one population sampled in the modern day (Population 30) has had half of its population replaced with lineages from a distant population (Population 1), simulating an admixture event.}\label{sfig:admixture_scenario}
\end{figure}

\gc{What do we want to say about the values of alpha used in matrix, I suspect we might want to average the frequencies over a number of draws of alpha from the posterior.} \gb{yeah good call.  right now I just use the last.params values, but I suppose we could either (a) use the MAP estimate, or (b), sample values of alpha from the joint posterior, calculate the f-stat for each draw, and average f-stats across all draws.}

We choose a focal population ($i$) that we wish to test. We remove this focal population from the dataset, and denote the dataset and covariance matrix where the $i^{th}$ population has been dropped with a ``$-i$'' in the subscript. We then predict the standardized allele frequencies at the geographic location ($G_i$) occupied by population $i$. To predict the standardized frequencies at  $G_i$, we use the mean frequency at $G_i$ under the conditional normal model. Given the parametric spatial covariance matrix $\Omega_{-i,-i}$, and the standardized allele frequencies $X_{-i}$ at all sampled geographic locations $G_{K-i}$, we calculate the conditional standardized observations at locus $\ell$ at location $G_i$ as
\begin{equation}
\label{eq:conditional_mean}
\bar{X}_{i,\ell} = \left(	\omega_{i,-i} \right) \left( \omega^{-1}_{-i,-i} \right) \left( X_{-i,\ell} \right),
%\bar{X}_i = \left(	\omega_{i,-i} \right) \left( \omega^{-1}_{-i,-i} \right) \left( X_{-i} \right),
\end{equation}
Note that we have described this in terms of the observed locations $G$ but we can equally well compute it using our inferred set of locations $G'$. 

%where we denote dropping the focal population $i$ from the covariance matrix or the observations $X$ at locus $\ell$ with a ``$-i$'' in the subscript.

\gc{Please use caps for matrices}

Then, given the conditional mean observations in focal population $i$, $\bar{X}_i$, and a population $j$ with which we wish to test for admixture with $i$, we calculate our spatial admixture statistic over all $L$ loci as follows:
\begin{equation}
\label{eq:spatial_fstat_pop}
f_{i,j}   = \frac{1}{C_{i,j}}	\mathbb E_L\left[(\vec{X}_{i,\ell}- \bar{X}_{i,\ell}) ~ (\vec{X}_{j,\ell})\right]	\\
\end{equation}
where $C_{i,j}$ is a normalization constant
\begin{equation}
C_{i,j}  = 	\sqrt{	\text{Var}( \vec{X}_i- \bar{X}_i )  \text{Var}( \vec{X}_{j} )  }  
\end{equation}
such that $0 \leq f_{i,j} \leq 1$ and can be interpreted as a correlation coefficient between $i$ and $j$, having regressed out our predicted frequencies at $i$.
\gc{Is this correct? maybe should be using mean here instead of expectation. I don't think you want these to be vectors }
This statistic calculates whether a focal population's deviations from its spatial predictions have excess covariance with another sampled population. The statistic $f_{i,j}$ is expected to be zero if the model has fully accounted for the covariance in allele frequencies between population $i$ and $j$. To identify significant departures from this expectation we can calculate a confidence interval for $f_{i,j}$ to see if it includes zero by using a non-parametric bootstrap \gb{or jackknife???} across loci in the dataset (or block bootstrap for linked data).  \gb{Figure 3a shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}

This statistic may also be calculated between a focal population $i$ and any arbitrary location $j'$ in space.  First, we calculate the spatial covariance between the arbitrary location $j'$ and all the sampled populations, \emph{excluding} the focal population $i$.  This spatial covariance matrix, which we denote $\Omega'$ , therefore has rank $k$, as we have excluded the focal population $i$ and included the putative source of admixture location $j'$.  Next, we calculate the conditional mean observations $X_{j'}$ at locus $\ell$ at location $G_{j'}$ as follows:
\begin{equation}
\label{eq:conditional_mean}
\bar{X}_{j',\ell} = \left(	\omega'_{-j',-j'} \right) \left( \omega'^{-1}_{-j',-j'} \right) \left( X_{-i,\ell} \right)
\end{equation}
where can use this in place of $X_{j,\ell}$ in equation \ref{eq:spatial_fstat_pop} to compute $f_{i,j'}$. 
By calculating $f_{i,j'}$ at points on a spatial grid, we can visualize the signature of excess covariance across the entire map. \gb{Figure 3b shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}

\gc{Comment on negative fs and intepretation?}

%We can compute $f_{i,j'}$ across a spatial grid to visualize where our 

% Finally, we can calculate a spatial $f$-statistic between the focal population and the putative location from which it receives admixture as follows,
% \begin{equation}
% \label{eq:spatial_fstat_location}
% f_{i,j'} = \frac{	\mathbb E_L\left[(\vec{X}_i- \bar{X}_i) ~ (\vec{\bar{X}}_{j'})\right]	}
% 			{\sqrt{	\text{Var}( \vec{X}_i- \bar{X}_i )  \text{Var}( \vec{\bar{X}}_{j'} )  }  }
% \end{equation}


This procedure allows us to visualize the signature of excess covariance across the entire map. \gb{Figure 3b shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}