\documentclass[12pt]{article}
\usepackage{geometry} 
\usepackage{graphicx}
\usepackage{float}
\usepackage{subcaption}
\usepackage{url}
\usepackage{mathtools}
\usepackage{color}
\usepackage{natbib}
\usepackage{fullpage}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{bigints}

% Make paragraphs not indented and leave a line between paragraphs
\setlength{\parskip}{\baselineskip}%
\setlength{\parindent}{0pt}%

\newcommand{\e}[1]{{\mathbb E}\left[ #1 \right]}
\newcommand{\gb}[1]{{\em \color{magenta} #1}}
\newcommand{\plr}[1]{{\em \color{green} #1}}
\newcommand{\gc}[1]{{\em \color{blue} #1}}

\geometry{a4paper}

\title{A Novel Spatial Framework for Understanding Genetic Admixture}
\date{\vspace{-5ex}}
\author{Gideon S. Bradburd$^{1,a}$, Peter L. Ralph$^{3,b}$, Graham M. Coop$^{1,c}$}

\begin{document}

\maketitle

\textsuperscript{1}Center for Population Biology, Department of Evolution and Ecology, University of California, Davis, CA 95616

\textsuperscript{3}Department of Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089

\textsuperscript{a}gbradburd@ucdavis.edu; 
\textsuperscript{b}pralph@usc.edu;
\textsuperscript{c}gmcoop@ucdavis.edu\\\\\

\newpage

\begin{abstract}
The patterns of genetic variation observed in modern populations are the product of a complex demographic and evolutionary history.  Genetic data can be used to illuminate that history, providing information about when and how populations have diverged, how migration connects populations, and how population sizes have fluctuated over time.  Work in this area has largely focused on estimating a population phylogeny, in which shared branch length on a tree represents shared evolutionary history between a pair of populations sampled in the modern day.  However, patterns of population differentiation are rarely tree-like, as migration and colonization will continuously re-shape patterns of relatedness between populations.  Isolation by distance (IBD), in which population differentiation increases with the distance between them, may offer a more natural null hypothesis.  Here, we present a novel analytical framework, SpaceMix, for the study of spatial genetic variation and genetic admixture, and a simple statistic to describe a population’s admixture status.
\end{abstract}

\newpage
%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Introduction}

Population-level demographic processes leave their mark on patterns of genetic variation and differentiation within and between populations.  Recent population genetics methods have focused on ways of learning about those processes from genetic observations, including demographic inference from the allele frequency spectrum (Song) and inference of the ancestral recombination graph from orthologous sequence data (Song, Rasmussen\&Siepel).  

Further work has focused on joint inference and visualization of whole-genome patterns of relatedness, specifically on estimating a population phylogeny describing the evolutionary history shared between sampled individuals or populations (e.g. Pickrell \& Pritchard, Patterson \& Moorjani, Reich?).  The approach of modeling relatedness between populations within species as a tree-like structure was pioneered by Cavalli-Sforza (\&Thompson?) in the 1960s \gb{?}, when genetic character data (blood types) were first becoming available in humans.  

Their work has been greatly extended by Pickrell and Pritchard (2012), whose method, TreeMix, models covariance in allele frequencies across loci as a directed acyclic graph between populations.  TreeMix then accommodates reticulate population structure by allowing branches on the population tree tree to be connected by arrows of admixture that explain excess residual covariance between populations or population 'clades.' 

In parallel, Patterson and Reich and Moorjani and Other Folks have developed a suite of tests for genetic admixture between a set of populations assuming a tree-like structure to their evolutionary history.  These tests model shared genetic drift as shared branch length on a hypothesized population tree, and take covariance in excess of that stipulated by the tree to be evidence for admixture.

These tree-based methods are both valuable as genetic inference and visualization tools.  However, a more natural framework for modeling genetic differentiation between sampled populations or individuals may be that of isolation by distance (IBD) (Wright).  The pattern of IBD, in which genetic differentiation increases with the geographic distance between populations, is ubiquitous in nature (Meirmans 2012, Ralph \& Coop 2012), and relies only on the weak assumption that individual's mating opportunities are geographically limited by dispersal.  

There is a rich history of this type of data visualization, especially using reduced dimensionality representations of the data such as Principal Components Analysis (e.g. Menozzi, Piazza, and Cavalli-Sforza 1978, Novembre et al 2008, Novembre and Stephens 2008).  Our method offers an improvement over PCA-based visualization methods in that it offers an explicit model of spatial genetic variation, and can therefore be used as a robust framework for inference and hypothesis testing (see also Yang et al 2012, Yang et al 2014).

%Isolation by distance is, in many ways, a more reasonable null hypothesis of population relatedness than a population phylogeny;  \gb{too compare-y?  should I ditch this?} population structure will only rarely be truly tree-like, and a strictly bifurcating graph is unable to accommodate many geographic scenarios, such as multiple equidistant populations in migration-drift equilibrium.

Here, we present an analytical framework and develop an inference algorithm for studying the spatial distribution of genetic variation based on a model of isolation by distance.  We also introduce a simple statistic for quantifying a population's admixture status, and demonstrate the utility of this approach with two high profile empirical applications.

%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Methods}
Throughout these Methods, we will introduce example scenarios, simulated under a neutral, spatially structured coalescent process.  We will also present the results of SpaceMix analyses run on those data, both to illustrate its utility and to clarify our presentation.  For all details of simulation procedure and analyses, please see Appendix X.

\subsection*{Data}
Our data consist of $L$ unlinked variable loci sampled across $K$ populations, as well as the geographic coordinates, denoted $G_k$, (latitude and longitude) at which those populations were sampled (although $G_k$ may be missing for some or all of the populations.  For convenience, we use bi-allelic single nucleotide polymorphisms (SNPs) as the genetic data in our descriptions below, but we note that this method could be applied equally well to other types of data, including microsatellites.  We refer to the $K$ accessions as populations, but the method can also be applied to individuals.  We summarize these genetic data as a set of allelic count data and sample size data, arbitrarily choosing an allele to count at each locus, and denoting the number of the counted allele at locus $\ell$ in population $k$ as $C_{\ell,k}$ out of a total sample size of $S_{\ell,k}$ alleles.  The sample frequency of the counted allele at locus $\ell$ in population $k$ is therefore $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  Our method models the covariance in these allele frequencies between populations across loci.


%%%%%%%%% %%%%%%%%% 
\subsection*{The Normal Approximation to Drift}
In order to model this covariance, we draw on previous work (Cavalli-Sforza \& Edwards, Nicholson et al 2002, Coop et al 2010, Pickrell \& Pritchard 2012), and model the process of genetic drift, \gc{i.e. the compouding of binomial sampling over the generations,}  as approximately Gaussian. Briefly, if the population frequency of allele $\epsilon$ in an ancestral population, $A$ is $\epsilon_A$, then we model the population frequency of that allele in a daughter population,  $B$, as follows:
\begin{equation}
\label{eq:normal_drift}
\epsilon_B \sim N(\epsilon_A,\delta_B(\epsilon_A)(1-\epsilon_A))
\end{equation}
where the parameter  $\delta_B$ is the amount of drift separating $A$ and $B$, a parameter shared by all neutral loci in the entire genome.  The binomial variance term in the normal variance in \eqref{eq:normal_drift} ignores the fact that the drift variance will change from generation to generation due to changes in the frequency of $\epsilon$ within a population over time.  This approximation will work best for alleles at intermediate frequency, and in situations where the extent of drift is sufficiently limited  (short time-scales or large population sizes). 

Among populations, the differentiating process of drift is counteracted by the homogenizing force of migration, so that populations with higher levels of historical or ongoing migration can be thought of as having more shared drift, i.e. stronger covariance in their allele frequency deviation around the ancestral (or global) mean. We model the population frequencies at a locus as multivariate normal (MVN) with mean $\epsilon$ and covariance matrix $\epsilon (1-\epsilon)\Omega$. The multivariate normal distribution offers a natural statistical framework for describing this covariance, which may be straightforwardly modeled as a parametric function of any pairwise distance variable (see, e.g., Bradburd, Ralph, and Coop 2013). 

% Unlike multivariate normal observations however, allele frequencies at variable loci are constrained to have a mean between 0 and 1, and may have heterogeneous variance across loci.  To more closely meet the assumptions of our statistical inference model, we mean-center and normalize our observations by the following procedure.

However, we do not have the population frequencies nor the `ancestral' frequency $\epsilon$. Instead we mean-center and normalize our observations at a locus using the weighted mean sample frequency in place of $\epsilon$.  Recall that the sample allele frequency at locus $\ell$ in population $k$ is given by $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  We wish to calculate a sample mean frequency at each locus weighted by the sample size in each population.  As sample size may vary across loci, we first calculate $\bar{S}_k$, the mean population sample size in population $k$, as $\bar{S}_k = \frac{1}{L}\sum_L S_{\ell,k}$.  We then calculate the weighted sample mean frequency at locus $\ell$ as follows:
\begin{equation}
\label{eq:sample_mean_freq}
\bar{f}_{\ell} = \frac{1}{\sum_K S_{\ell,k}} \sum_K \hat{f}_{\ell,k} S_{\ell,k}
\end{equation}
We approximate the binomial variance at each locus by $\bar{f}_{\ell}(1-\bar{f}_{\ell})$.  To avoid modeling the heterogeneous effect of this variance across loci, we standardize by this variance.  We call the standardized allele frequencies $X_{\ell,k}$, and calculate them as follows:
\begin{equation}
\label{eq:MCN_freqs}
X_{\ell,k} = \frac{ \hat{f}_{\ell,k} - \bar{f}_{\ell} } {\bar{f}_{\ell}(1-\bar{f}_{\ell})}
\end{equation}
Note, by using the sample mean frequency to mean-center our observations, we lose a degree of freedom, and reduce the covariance across loci between populations (sometimes inducing negative covariance among distant populations). We accomodate the extra sampling noise distortion, and reduced rank of the covariance matrix by assuming that our
\begin{equation}
X_{\ell} \sim MVN(0, \Omega^{\prime} )
\end{equation}
where $\Omega^{\prime}$ is a simple transform of $\Omega$. We discuss this further in Appendix A.

\gc{Okay so I think the right strategy re. the mean centering and dropping is treat everything in the main text like we've got the global freq. and then just apply that twist in the appendix}

%Note also that this approximation works best for alleles at intermediate frequency, which are less likely to be lost or fixed during the interval separating $A$ and $B$.  However, if the extent of drift sufficiently limited (time scales are short or population sizes are large), the approximation of inter-generational binomial sampling to Brownian motion is good

%The parameter $\delta_B$, the amount of drift that has occurred in $B$ since it split off from $A$, is approximately equal to $\frac{t}{2N_e}$, where $t$ is time in generations, and $N_e$ is the effective population size.  Note that the binomial variance term in the normal variance in \eqref{eq:normal_drift} ignores the fact that the drift variance will change from generation to generation due to changes in the frequency of $\epsilon$ within a population over time.  Note also that this approximation works best for alleles at intermediate frequency, which are less likely to be lost or fixed during the interval separating $A$ and $B$.  However, if the extent of drift sufficiently limited (time scales are short or population sizes are large), the approximation of inter-generational binomial sampling to Brownian motion is good (\gb{see figure comparing fit of binomial sampling to fit of normal approx?}).  

% Between populations, the differentiating process of drift is counteracted by the homogenizing force of migration, so that populations with higher levels of historical or ongoing migration can be thought of as having more shared drift.  Across loci, populations with highly shared drift will tend to covary more strongly in the deviations their allele frequencies take from some ancestral (or global) mean.

% %%%%%%%%% %%%%%%%%% 
% \subsection*{Mean-centering and normalizing variance}
% The multivariate normal distribution offers a natural statistical framework for describing this covariance, which may be straightforwardly modeled as a parametric function of any pairwise variable (see, e.g., Bradburd, Ralph, and Coop 2013).  Unlike multivariate normal observations however, allele frequencies at variable loci are constrained to have a mean between 0 and 1, and may have heterogeneous variance across loci.  To more closely meet the assumptions of our statistical inference model, we mean-center and normalize our observations by the following procedure.

% Recall that the sample allele frequency at locus $\ell$ in population $k$ is given by $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  We wish to calculate a sample mean frequency at each locus weighted by the sample size in each population.  As sample size may vary across loci, we first calculate $\bar{S}_k$, the mean population sample size in population $k$, as $\bar{S}_k = \frac{1}{\ell}\sum_\ell S_{\ell,k}$.  We then calculate the weighted sample mean frequency at locus $\ell$ as follows:
% \begin{equation}
% \label{eq:sample_mean_freq}
% \bar{f}_{\ell} = \frac{1}{\sum_k S_{\ell,k}} \sum_k \hat{f}_{\ell,k} S_{\ell,k}
% \end{equation}
% The binomial variance at each locus varies as a function of the global allele frequency, and, at locus $\ell$, is given by $\bar{f}_{\ell}(1-\bar{f}_{\ell})$.  To avoid modeling the heterogeneous effect of this variance across loci, we standardize by the binomial variance.  We call the mean-centered normalized allele frequencies $X_{\ell,k}$, and calculate them as follows:
% \begin{equation}
% \label{eq:MCN_freqs}
% X_{\ell,k} = \frac{ \hat{f}_{\ell,k} - \bar{f}_{\ell} } {\bar{f}_{\ell}(1-\bar{f}_{\ell})}
% \end{equation}
% We then model these transformed data as multivariate normal, as described in the next section. 



%%%%%%%%% %%%%%%%%% 
\paragraph{Spatial Covariance Model}
We wish to model the covariance between populations as the result of a spatial process, in which migration rates between nearby populations are higher than between distant ones, so that a population has higher covariance with a close neighbor than with a more distant population.  We choose as the form of our parametric covariance matrix a simple and flexible model, with an exponential decay of allele frequency covariance with geographic distance (Wasser et al 2004, Bradburd, Ralph, and Coop 2013).  The covariance between populations $i$ and $j$ is proportional to 
\begin{equation}
\label{eq:spatial_covariance}
\Omega_{i,j} = \frac{1}{\alpha_0} \text{exp} \left(	\left( \alpha_1D_{i,j} \right)^{\alpha_2} \right) + I\bar{S_k}^{-1} + I\eta_k \text{,}
\end{equation}
where $D_{i,j}$ is the geographic distance between population $i$ and $j$ (and therefore a function of their locations, $G_i$ and $G_j$), $\alpha_0$ controls the within-population variance, or the covariance when distance between points is 0 (the sill of the covariance matrix),  $\alpha_1$ controls the rate of the decay of covariance per unit pairwise distance, and $\alpha_2$ determines the shape of that decay.  In addition, we introduce population-specific variance terms to accommodate unshared drift and sample size effects.  These are given by $I\bar{S_k}^{-1} + I\eta_k$, where $I$ is the identity matrix, $I\bar{S_k}$ is the mean sample size in population $k$ across all loci, and $\eta_k$ is the nugget estimated in population $k$.  

This matrix $\Omega$ can be transformed into $\Omega^{\prime}$, the standardized sample frequency covariance matrix.
%\gc{what about the nugget?} \gb{it's complicated and a little confusing, because, as incorporated into the admixed covariance below, this covariance function doesn't get a nugget (that would be double-dipping the nugget).  So I'm not sure the best way to introduce the nugget.  Clearly, it ought to be here for the non-admixed model, so maybe we should introduce yet \emph{another} variable name, which would be the spatial covariance \emph{sans} nugget, and then use that in the admixed form of $\Omega$ below?}


Given the assumption of multivariate normality for our sample frequencies, it follows that the sample covariance of our standardized sample frequencies calculated across loci ($\widehat{\Omega} = X X^T$)  is Wishart distributed with degrees of freedom equal to the number of loci ($L$) across which the covariance is calculated.
That is, 
\begin{equation}
\label{eq:wishart_dist}
\widehat{\Omega} \sim \mathcal{W}\left( L^{-1} \Omega^{\prime}, L	\right)
\end{equation}
\gc{should the $L^{-1}$ be there? Need to work a bit to get this right in terms of transform, but I think this is the right way to go.} So we can estimate the parameters of our simple isolation by distance model, $\alpha_0,~\cdots, \alpha_2$, by treating \ref{eq:wishart_dist} as the likelihood of our standardized sample frequencies across loci, as this contains all of the information about our parameters of interest. \gc{Mention prior on params make this posterior?} Handily it also means that once the sample covariance matrix has been calculated all other computations do not scale linearly with number of loci, making the method scalable to genome size datasets. \gc{Have some link to an appendix describing what to do with linked loci?}


\gc{MOVE to appendix, or delete.
The likelihood of the data $X_{\ell,k}$, given the observed distance matrix $D$ between all $K$ populations and the values of the three $\alpha$ parameters ($\vec{\alpha}$), which together parameterize the spatial covariance matrix $\omega$, is given by
\begin{equation}
\label{eq:wishart_lnl}
\mathcal{P}(X_{\ell,k} \; | \; D(G), \pmb{\vec{\alpha}}) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\omega}^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\pmb{\omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}
where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface variables are parameters estimated as part of our inference procedure.
}


%%%%%%%%% %%%%%%%%% 
\paragraph{Accommodating non-equilibrium processes}
%\gb{MS simulation of pops on a grid, figure to show that we recover the grid}
%\gb{MS simulation of pops on a grid with a barrier in the middle, figure to show how pops on either side of barrier clump together (or how pops scuttle away from each other)}

\gb{MS simulation of pops on a grid with a recent expansion, figure to show how those pops all clump toward source of expansion}

This model assumes that the variance in allele frequencies is the same in all locations and that the covariance between pairs of populations decays in the same way with geographic distance in all portions of the sampled range (i.e. that the process is homogeneous and isotropic). However, in many cases those assumptions will not be met. Non-equilibrium processes like long distance admixture, colonization, or population expansion events will distort the relationship between covariance and distance across the range.  Barriers to dispersal on the landscape can also change the relationship between allele frequency covariance and distance.

%bias the estimation of the contribution of a unit distance to decay in covariance.

%\gc{got rid of bias parameter estimates bits, as it's not clear what these parameters are anyway, esp. if model is very wrong}
%\gc{Or could intro. nugget here, as part of non-equilibrium}

One way that we can accommodate these heterogeneous processes is to allow populations to choose their own locations with respect to each other to maximize the resemblance to isolation by distance. Two populations that are sampled at distant locations but that are genetically similar (perhaps one was recently founded by a colonization event from the other) may choose estimated locations that are nearby, while two populations that are sampled close together, but that are genetically dissimilar (e.g, are separated by a barrier), may choose locations that are farther apart. The result is a \gc{``geogenetic''} map in which the distances between populations are indicative of the way that populations perceive the distances between themselves.  

To do this we can treat population's geographic locations as a random variable, and estimate them as part of our bayesian inference procedure.  We write our posterior as 
\begin{equation}
P(\pmb{G'}, \pmb{\vec{\alpha}}| X ) \propto  P(\widehat{\Omega} |\Omega (\vec{\alpha},\pmb{D}(\pmb{G'}) ) P(\pmb{\vec{\alpha}}) P(\pmb{G'})  
\end{equation}
where the constant of proportionality is the normalization constant of the posterior. The probability of the sample covariance matrix, $P(\widehat{\Omega} |\Omega (\vec{\alpha}\pmb{D}(\pmb{G'}) )$, is as before the Wishart likelihood of 
the sample covariance matrix of our standardized frequencies, and our priors on $\pmb{\vec{\alpha}}$ are as before. Here $P(\pmb{G'}) $ are the priors for the spatial locations, which we assume are independent across locations. For the prior on population $k$'s estimated location, $G'_k$, we use a bivariate normal spatial prior centered on the observed location $G_k$. We perform MCMC on all of the random variables; for more details on our Bayesian inference procedure see Appendix B. 
 
The sampled locations are a natural prior on where the populations are positioned under the null of isolation by distance (as well as a natural starting position for the MCMC). This prior also encourages the resulting inferred geogenetic map to be anchored in the observed locations and to represent (informally) the minimum distortion to space necessary to satisfy constraints placed by genetic similarities of populations. However, one concern is that if the method returns a spatial configuration that strongly resembles a map, this could reflect the possibility that there is insufficient information in the data to overcome the influence of the prior, or that the MCMC having not been run long enough.  To address this concern we can also use random locations as the ``observed locations" ($G_k$) to remove any influence of the observed map on the output via the prior, or we can change the variance on the spatial priors to ascertain the effect of the prior on inference.

%remove ir more to approxim
% Our likelihood function is now given by,
% \begin{equation}
% \label{eq:choose_location_wishart_lnl}
% \mathcal{P}(X_{\ell,k} \; | \; \pmb{D}(\pmb{G'}), \pmb{\vec{\alpha}}) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\omega}^{-1}Y)}{2}	\right) }	
% 									{2^{\frac{LK}{2}}  |\frac{\pmb{\omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
% \end{equation}
% where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface variables are parameters estimated as part of our inference procedure.  Notice that this equation has the same form as \eqref{eq:wishart_lnl}, save that the pairwise geographic distance matrix $D$ is now a parameter to be estimated rather than an observed value.

To illustrate this inference procedure, we present several scenarios simulated under the coalescent (using \textrm{ms}, (Hudson)), featured in Figure \ref{sfig:lattice_scenarios}, as well as the output of SpaceMix analyses run on them.  In the first scenario (Fig. \ref{sfig:lattice_scenarios}\subref{lattice_pops}), we simulate a stepping stone model, with symmetric nearest neighbor migration.  In our second scenario (Fig. \ref{sfig:lattice_scenarios}\subref{barrier_pops}), we simulate under the same stepping stone model, but introduce a barrier to dispersal, across which migration rates are attenuated by a factor of 5.  On both scenarios, we perform SpaceMix analyses in which we treat population locations as random variables to be estimated as part of the model, and randomly place the `observed' location of each population on a line so as to remove the influence of the prior.

%% Perhaps just say this in the methods section (attenuated by $\sqrt{2}$ on the diagonals) and include file of ms command lines. 

\begin{figure}
	\centering
		\subcaptionbox{Populations on a lattice \label{lattice_pops}}
			{\includegraphics[width=2.4in,height=2in]{stationary_pops_map.png}}
		\subcaptionbox{SpaceMix map \label{lattice_pops_inference}}
			{\includegraphics[width=2.4in,height=2in]{stationary_pops_1_inference_map.png}}
		\subcaptionbox{A lattice with a barrier \label{barrier_pops}}
			{\includegraphics[width=2.4in,height=2in]{barrier_map.png}}
		\subcaptionbox{SpaceMix map \label{barrier_pops_inference}}
			{\includegraphics[width=2.4in,height=2in]{barrier_1_inference_map.png}}
	\caption{Simulated scenarios and the corresponding population maps inferred using SpaceMix.  For clarity of presentation, the inferred coordinates have have been Procrustes transformed around the coordinates of the lattice used to simulate the data.}\label{sfig:lattice_scenarios}
\end{figure}

In Figure \ref{sfig:lattice_scenarios}\subref{lattice_pops_inference}, the reader can see that the configuration estimated for the populations by SpaceMix matches the lattice structure used to simulate the data, and that populations are correctly choosing their nearest neighbors.  Similarly, in Figure \ref{sfig:lattice_scenarios}\subref{barrier_pops_inference}, the population configuration matches that of the lattice used to simulate the data, but the reader can see that, due to the influence of the barrier, the two halves of the map have pushed farther away from one another.  This gap between populations on either side of the barrier reflects the way those populations perceive the increased effective distance between them. \gc{Perhaps give the covariance vs distance plots in supplement.}

\gc{I'd perhaps introduce the spatial admixture simulations here. That would motivate the admixture stats}

%%%%%%%%% %%%%%%%%% 
\subsection*{Spatial Admixture Statistic}

\gc{Do we also want to describe comparison of observed to expected covariance matrix?? Guess we could do it above, to show that our model has successfully improved the fit to the data.}

 
\gb{Figure 2: MS simulation of pops on a grid with 1 admixture event from a pop on one side to a pop on the other}

\gb{3a figure to show the pairwise f-stat between all pops and the admixed pop}

\gb{3b figure to show landscape of admixture to the admixed pop}

If our data are well fit by a model of isolation by distance then (1) a population's genetic makeup should be well predicted by that of its neighbors, and (2) populations should not show excess covariance with distant populations. Violation of either of these two points will result in a poor fit of a simple isolation by distance model, and particularly a violation of point (2) may indicate that long distance admixture has taken place. We develop simple statistics, which we call spatial $f$-statistics \gb{should we give these a real name?}, to test these points \gc{do we actually test 1?}.

These statistics are in the spirit of both the $f_n$ statistics introduced by Reich et al (2009) and the admixture arrows in a TreeMix graph (Pickrell and Pritchard 2012), and are designed to pick up a signal of covariance between populations in excess of that predicted by our model.  These previous approaches use a null model of a branching population phylogeny and develop tests to diagnose covariance between populations greater than that predicted by the population tree.  Specifically, the tests identify violations of conditional independence of allele frequencies among populations that indicate that there has been admixture (gene flow) between disparate parts of the tree.  In place of a phylogeny, our tests use a spatial model of allelic covariance to look for population pairs whose sample covariance exceeds that predicted by their observed distance and the inferred parameters of the spatial model. %excess covariance between distant populations, indicative of excess gene flow.

%in which one population's covariance with another is expected to decay at a fixed manner across the landscape, as determined by the estimated parametric form of the spatial covariance.  

The details of this procedure are given below, and illustrated with a simulated example scenario (see Figure \ref{sfig:admixture_scenario}), featuring an admixture event.

\begin{figure}
	\centering
	\includegraphics[width=2.4in,height=2in]{admixture_map.png}
	\caption{In this simulated scenario, one population sampled in the modern day (Population 30) has had half of its population replaced with lineages from a distant population (Population 1), simulating an admixture event.}\label{sfig:admixture_scenario}
\end{figure}

\gc{What do we want to say about the values of alpha used in matrix, I suspect we might want to average the frequencies over a number of draws of alpha from the posterior.} \gb{yeah good call.  right now I just use the last.params values, but I suppose we could either (a) use the MAP estimate, or (b), sample values of alpha from the joint posterior, calculate the f-stat for each draw, and average f-stats across all draws.}

We choose a focal population ($i$) that we wish to test. We remove this focal population from the dataset, and denote the dataset and covariance matrix where the $i^{th}$ population has been dropped with a ``$-i$'' in the subscript. We then predict the standardized allele frequencies at the geographic location ($G_i$) occupied by population $i$. To predict the standardized frequencies at  $G_i$, we use the mean frequency at $G_i$ under the conditional normal model. Given the parametric spatial covariance matrix $\Omega_{-i,-i}$, and the standardized allele frequencies $X_{-i}$ at all sampled geographic locations $G_{K-i}$, we calculate the conditional standardized observations at locus $\ell$ at location $G_i$ as
\begin{equation}
\label{eq:conditional_mean}
\bar{X}_{i,\ell} = \left(	\omega_{i,-i} \right) \left( \omega^{-1}_{-i,-i} \right) \left( X_{-i,\ell} \right),
%\bar{X}_i = \left(	\omega_{i,-i} \right) \left( \omega^{-1}_{-i,-i} \right) \left( X_{-i} \right),
\end{equation}
Note that we have described this in terms of the observed locations $G$ but we can equally well compute it using our inferred set of locations $G'$. 

%where we denote dropping the focal population $i$ from the covariance matrix or the observations $X$ at locus $\ell$ with a ``$-i$'' in the subscript.

\gc{Please use caps for matrices}

Then, given the conditional mean observations in focal population $i$, $\bar{X}_i$, and a population $j$ with which we wish to test for admixture with $i$, we calculate our spatial admixture statistic over all $L$ loci as follows:
\begin{equation}
\label{eq:spatial_fstat_pop}
f_{i,j}   = \frac{1}{C_{i,j}}	\mathbb E_L\left[(\vec{X}_{i,\ell}- \bar{X}_{i,\ell}) ~ (\vec{X}_{j,\ell})\right]	\\
\end{equation}
where $C_{i,j}$ is a normalization constant
\begin{equation}
C_{i,j}  = 	\sqrt{	\text{Var}( \vec{X}_i- \bar{X}_i )  \text{Var}( \vec{X}_{j} )  }  
\end{equation}
such that $0 \leq f_{i,j} \leq 1$ and can be interpreted as a correlation coefficient between $i$ and $j$, having regressed out our predicted frequencies at $i$.
\gc{Is this correct? maybe should be using mean here instead of expectation. I don't think you want these to be vectors }
This statistic calculates whether a focal population's deviations from its spatial predictions have excess covariance with another sampled population. The statistic $f_{i,j}$ is expected to be zero if the model has fully accounted for the covariance in allele frequencies between population $i$ and $j$. To identify significant departures from this expectation we can calculate a confidence interval for $f_{i,j}$ to see if it includes zero by using a non-parametric bootstrap \gb{or jackknife???} across loci in the dataset (or block bootstrap for linked data).  \gb{Figure 3a shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}

This statistic may also be calculated between a focal population $i$ and any arbitrary location $j'$ in space.  First, we calculate the spatial covariance between the arbitrary location $j'$ and all the sampled populations, \emph{excluding} the focal population $i$.  This spatial covariance matrix, which we denote $\Omega'$ , therefore has rank $k$, as we have excluded the focal population $i$ and included the putative source of admixture location $j'$.  Next, we calculate the conditional mean observations $X_{j'}$ at locus $\ell$ at location $G_{j'}$ as follows:
\begin{equation}
\label{eq:conditional_mean}
\bar{X}_{j',\ell} = \left(	\omega'_{-j',-j'} \right) \left( \omega'^{-1}_{-j',-j'} \right) \left( X_{-i,\ell} \right)
\end{equation}
where can use this in place of $X_{j,\ell}$ in equation \ref{eq:spatial_fstat_pop} to compute $f_{i,j'}$. 
By calculating $f_{i,j'}$ at points on a spatial grid, we can visualize the signature of excess covariance across the entire map. \gb{Figure 3b shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}

\gc{Comment on negative fs and intepretation?}

%We can compute $f_{i,j'}$ across a spatial grid to visualize where our 

% Finally, we can calculate a spatial $f$-statistic between the focal population and the putative location from which it receives admixture as follows,
% \begin{equation}
% \label{eq:spatial_fstat_location}
% f_{i,j'} = \frac{	\mathbb E_L\left[(\vec{X}_i- \bar{X}_i) ~ (\vec{\bar{X}}_{j'})\right]	}
% 			{\sqrt{	\text{Var}( \vec{X}_i- \bar{X}_i )  \text{Var}( \vec{\bar{X}}_{j'} )  }  }
% \end{equation}


This procedure allows us to visualize the signature of excess covariance across the entire map. \gb{Figure 3b shows the spatial \emph{f}-stat calculated between the admixed population and each other population.}

%%%%%%%%% %%%%%%%%% 
\subsection*{Inference of Spatial Admixture}

This signal of excess covariance over anomalously long distances can be difficult to accommodate within the framework described above, in which we estimate the 2-dimensional configuration of populations that, along with a model of the decay of covariance in allele frequencies with distance, best describes their empirical patterns of genetic differentiation.  In Figure \ref{sfig:admixture_inference_target_loc}, the reader can see the torturous lengths to which the method goes to come up with a configuration of populations that accommodates their genetic relationships.  The admixed population 30 is estimated to have a location intermediate between population 1, the source of its admixture, and populations 24, 25, and 29, the nearest neighbors to the location of its non-admixed lineages.  However, this warping of space is difficult to interpret, and does not have great utility for the visualization of genetic relationships.

\begin{figure}[ht!]
	\centering
	\includegraphics[width=2.4in,height=2in]{admixture_1_inference_map.png}
	\caption{Inference of population locations in the scenario depicted in Figure \ref{sfig:admixture_scenario}.  Population 30 has received half of its lineages from population 1, to simulate a long distance admixture event in the very recent past.}\label{sfig:admixture_inference_target_loc}
\end{figure}

Instead, we can interpret this excess covariance over anomalously long distances as genetic admixture, and we can incorporate it directly into out inference framework.  We define the allele frequency at locus $\ell$ in admixed population $k$ to be
\begin{equation}
f_{\ell,k} = pf_{\ell,i} + (1-p)f_{\ell,j},
\end{equation}
where $f_{\ell,i}$ is the allele frequency in population $i$, $f_{\ell,j}$ is the allele frequency in population $j$, and $p$ is the admixture proportion, which varies between 0 and 1 and describes the extent to which populations $i$ and $j$ are contributing to the genetic make-up of population $k$.

To infer the spatial context of this admixture, we allow each population a point in space, which we refer to as its source of admixture, from which it draws its admixture, and we model both the location of that source and the extent (proportion) of that admixture.  The observed allele frequencies in sampled populations are therefore a weighted average of the model-estimated allele frequencies at the geographic location of the sampled population and those at the coordinates of the source from which the observed population draws admixture.  That is, the observed allele frequencies in population $k$ are modeled as follows:
\begin{equation}
f_{k} = pf_{k'} + (1-p)f_{j},
\end{equation}
where $f_{k'}$ are the model-estimated allele frequencies across loci at the spatial location of population $k$ and $f_{j}$ are the model-estimated allele frequencies at the spatial location of the source of admixture $j$, from which population $k$ is drawing admixture in proportion $p$.  The admixture proportion $p$ is constrained to vary between 0 and 0.5, such that at least half of a population's genetic make-up must be determined by its geographic location.

The covariance between sampled populations $i$ and $j$ can be modeled as
\begin{alignat}{3}
\label{eq:admixed_covariance_1}
\Omega_{i,j} = (1-p_i)(1-p_j) \omega_{i\;,\;j\;} \; \times&\\
(p_i)(1-p_j) \omega_{i',\;j\;} \; \times   \notag&\\
(p_j)(1-p_i) \omega_{i\;,\;j'} \; \times   \notag&\\
(p_i)(p_j) \omega_{i',\;j'} \; \phantom{\times}   \notag&
\end{alignat}
where $i'$ and $j'$ are the points from which populations $i$ and $j$ are drawing their admixture with proportions $p_i$ and $p_j$, and $\omega$ is the spatial covariance function parameterized by the pairwise geographic distances between each pair of populations, \emph{without} the population-specific variance terms on the diagonal (i.e., in Eqn. \ref{eq:admixed_covariance_1}, $\omega_{i,j} = \frac{1}{\alpha_0} \text{exp} \left(	\left( \alpha_1D_{i,j} \right)^{\alpha_2} \right)$)

We re-introduce the population-specific variance terms on each diagonal element of this admixed covariance matrix.  The full expression for our admixed covariance function is below.
\begin{alignat}{3}
\label{eq:admixed_covariance_2}
\Omega_{i,j} = (1-p_i)(1-p_j) \omega_{i\;,\;j\;} \; \times&\\
(p_i)(1-p_j) \omega_{i',\;j\;} \; \times   \notag&\\
(p_j)(1-p_i) \omega_{i\;,\;j'} \; \times   \notag&\\
(p_i)(p_j) \omega_{i',\;j'} \; +   \notag&\\
I\bar{S_k}^{-1} + \eta_k \phantom{+} \notag&
\end{alignat}

where $I$ is the identity matrix, $\bar{S_k}$ is the mean sample size in population $k$ across all loci, and $\eta_k$ is the nugget estimated in population $k$.

The location of the source of admixture for population $k$, $G_{k'}$, and the population's admixture proportion, $p_k$, are treated as random variables and jointly estimated as part of our inference procedure.  The admixed covariance between populations $i$ and $j$, $\Omega_{i,j}$ is then a function of all the pairwise spatial covariances between populations $i$ and $j$ and the points from which they draw admixture, $i'$ and $j'$.  Those spatial covariances in turn are a function of all combinations of pairwise distances between their locations: $G_i$, $G_i$, $G_{i'}$, and $G_{j'}$.  This parametric covariance form is illustrated in Figure \ref{sfig:admixed_cov_diagram}.

\begin{figure}[ht!]
	\centering
	\includegraphics[width=2in,height=2in]{admix_cov_fig.pdf}
	\caption{An illustration of the form of the admixed covariance given in Eqns. \ref{eq:admixed_covariance_1} and \ref{eq:admixed_covariance_2}.  Populations $i$ and $j$ are drawing admixture in proportions $p_i$ and $p_j$ from their respective sources of admixture, $i'$ and $j'$, and all pairwise spatial covariances (the $\omega$'s) are shown.  In this cartoon example, population $j$ is drawing more admixture from its source $j'$ than $i$ is from its source $i'$ (i.e., $p_j > p_i$).}\label{sfig:admixed_cov_diagram}
\end{figure}


The likelihood function for this model in which each population is allowed to draw admixture from a point in space is 
\begin{equation}
\label{eq:choose_admixture_wishart_lnl}
\mathcal{P}\left(X_{\ell,k} \; | \; \pmb{D}\left(G,\pmb{G_{'}}\right), \pmb{\vec{\alpha}}, \pmb{p}, S, \pmb{\eta}\right) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\Omega}^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\pmb{\Omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}
where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function, and all boldface variables are parameters estimated as part of our inference procedure.

Using the example scenario described above and diagrammed in Figure \ref{sfig:admixture_scenario}, we demonstrate the inference of populations' sources and strengths of admixture and illustrate the results in Figure \ref{sfig:admixture_inference_source_loc}.  The reader can see that the only admixed population (Population 30) is drawing admixture from the location of the source of admixture that was used to simulate the data, and that all other populations, which are not admixed, are not choosing to draw admixture in non-negligible amounts.

\begin{figure}[ht!]
	\centering
	\includegraphics[width=2.4in,height=2in]{admixture_2_inference_map.png}
	\caption{Posterior distribution of inference of the sources and strengths of admixture for the sampled populations.  The admixed population (Population 30) is drawing admixture from the location of its source of admixture that was used to simulate the data (the location of Population 1).}\label{sfig:admixture_inference_source_loc}
\end{figure}

\subsection*{Models}
The models described above may be used in various combinations.  In the simplest model, populations may not choose their own locations, nor are they allowed to draw admixture, and the only parameters to be estimated are those of the spatial covariance function given in Eqn \eqref{eq:spatial_covariance}, and the population-specific variance terms ($\eta_k$).  In the most complex model, population locations, the locations of their sources of admixture, and the proportions of that admixture are all estimated jointly in addition to the parameters of the spatial covariance function and the population specific variances.  The full likelihood of this most complex parameterization is given by,
\begin{equation}
\label{eq:source_and_target_wishart_lnl}
\mathcal{P}\left(X_{\ell,k} \; | \; \pmb{D}\left(\pmb{G'},\pmb{G_{'}}\right), \pmb{\vec{\alpha}}, \pmb{p}, S, \pmb{\eta}\right) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\pmb{\Omega}^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\pmb{\Omega}}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}


\gb{A note about here how this complex model is still identifiable}

\section*{Inference}
For details on our Bayesian inference framework and Markov chain Monte Carlo inference procedure, please see the Section: How I spent the past year!

\section*{Empirical Applications}
To demonstrate the applications of this novel method, we employed it in two canonical empirical systems: the greenish warbler ring species complex, and, to our knowledge, the most complete geographic sampling of human populations to date (\gb{is this true?}).

\subsection*{Greenish Warblers}
The greenish warbler (\emph{Phylloscopus trochiloides}) species complex is a canonical proposed examples of a ring species (Irwin et al 2001).  The complex is broadly distributed around the Tibetan plateau, and exhibits gradients in both phenotype and call, as well as in genetic makeup (Irwin et al 2001, Irwin et al 2005, Irwin et al 2008).  At the northern end of the ring in central Siberia, where the eastern and western arms of population expansion meet, there are discontinuities in call and phenotype, and also reproductive isolation (Irwin et al 2001, Irwin et al 2008).  

Integral to the question of whether it constitutes a ring species in the purest  form of the idea is the issue of whether gene flow along the margins of the plateau has truly been continuous throughout the history of the expansion or if, alternatively, discontinuities in migration around the species complex's range have facilitated periods of differentiation in genotype or phenotype without gene flow (Mayr 1942, Coyne and Orr 2004).  Alcaide et al (2014), have suggested that the greenish warbler species complex constitutes a `broken' ring species, in which there have been historical discontinuities in gene flow that facilitated the evolution of reproductive isolation between adjacent forms.  Because the questions in this system are fundamentally both geographic and genetic in nature, it is eminently SpaceMix-able, and, within this spatial framework, we performed a number of analyses to investigate the geographic context of population differentiation in the greenish warbler species complex.

%At stake is whether the species offers an example of sympatric speciation (i.e. speciation with continuous gene flow).

For these analyses, we used the dataset from Alcaide et al (2014), which consisted of 95 individuals sampled at 22 distinct locations and sequenced at XXX SNPs.  

Details of analyses run
\subsection*{Human Populations}
Description of the data

Description of the research questions we use SpaceMix to address

Details of analyses run

%%%%%%%%% %%%%%%%%% 
\section*{Results}
\gb{should we just merge the results section in with the empirical applications section?  not sure what else would go here.}

%%%%%%%%% %%%%%%%%% 
\section*{Discussion}
In this paper we have presented blah blah blah.  We believe this represents an advance over previous methods because blah blah blah.  This method can be used to answer a variety of empirical questions, including blah, blah, and blah, and also serves as an intuitive data visualization tool.

%%%%%%%%% %%%%%%%%% 
\subsection*{Empirical Results}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Greenish Warblers}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Humans}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\section*{Future Directions}

spatiotemporal model

spatialSTRUCTURE
%%%%%%%%%





\end{document}













