\documentclass[12pt]{article}
\usepackage{geometry} 
\usepackage{graphicx}
\usepackage{float}
\usepackage{subfig}
\usepackage{url}
\usepackage{mathtools}
\usepackage{color}
\usepackage{natbib}
\usepackage{fullpage}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{bigints}

% Make paragraphs not indented and leave a line between pararaphs
\setlength{\parskip}{\baselineskip}%
\setlength{\parindent}{0pt}%

\newcommand{\e}[1]{{\mathbb E}\left[ #1 \right]}
\newcommand{\gb}[1]{{\em \color{magenta} #1}}
\newcommand{\plr}[1]{{\em \color{green} #1}}
\newcommand{\gc}[1]{{\em \color{blue} #1}}

\geometry{a4paper}

\title{A Novel Spatial Framework for Understanding Genetic Admixture}
\date{\vspace{-5ex}}
\author{Gideon S. Bradburd$^{1,a}$, Peter L. Ralph$^{3,b}$, Graham M. Coop$^{1,c}$}

\begin{document}

\maketitle

\textsuperscript{1}Center for Population Biology, Department of Evolution and Ecology, University of California, Davis, CA 95616

\textsuperscript{3}Department of Molecular and Computational Biology, University of Southern California, Los Angeles, CA 90089

\textsuperscript{a}gbradburd@ucdavis.edu; 
\textsuperscript{b}pralph@usc.edu;
\textsuperscript{c}gmcoop@ucdavis.edu\\\\\

\newpage

\begin{abstract}
The patterns of genetic variation observed in modern populations are the product of a complex demographic and evolutionary history.  Genetic data can be used to illuminate that history, providing information about when and how populations have diverged, how migration connects populations, and how population sizes have fluctuated over time.  Work in this area has largely focused on estimating a population phylogeny, in which shared branch length on a tree represents shared evolutionary history between a pair of populations sampled in the modern day.  However, patterns of population differentiation are rarely tree-like, as migration and colonization will continuously re-shape patterns of relatedness between populations.  Isolation by distance (IBD), in which population differentiation increases with the distance between them, may offer a more natural null hypothesis.  Here, we present a novel analytical framework, SpaceMix, for the study of spatial genetic variation and genetic admixture, and a simple statistic to describe a population’s admixture status.
\end{abstract}

\newpage
%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Introduction}

Population-level demographic processes leave their mark on patterns of genetic variation and differentiation within and between populations.  Recent population genetics methods have focused on ways of learning about those processes from genetic observations, including demographic inference from the allele frequency spectrum (Song) and inference of the ancestral recombination graph from orthologous sequence data (Song, Rasmussen\&Siepel).  

Further work has focused on joint inference and visualization of whole-genome patterns of relatedness, specifically on estimating a population phylogeny describing the evolutionary history shared between sampled individuals or populations (e.g. Pickrell \& Pritchard, Patterson \& Moorjani, Reich?).  The approach of modeling relatedness between populations within species as a tree-like structure was pioneered by Cavalli-Sforza (\&Thompson?) in the 1960s \gb{?}, when genetic character data (blood types) were first becoming available in humans.  

Their work has been greatly extended by Pickrell and Pritchard (2012), whose method, TreeMix, models covariance in allele frequencies across loci as a directed acyclic graph between populations.  TreeMix then accommodates reticulate population structure by allowing branches on the population tree tree to be connected by arrows of admixture that explain excess residual covariance between populations or population 'clades.' 

In parallel, Patterson and Reich and Moorjani and Other Folks have developed a suite of tests for genetic admixture between a set of populations assuming a tree-like structure to their evolutionary history.  These tests model shared genetic drift as shared branch length on a hypothesized population tree, and take covariance in excess of that stipulated by the tree to be evidence for admixture.

These tree-based methods are both valuable as genetic inference and visualization tools.  However, a more natural framework for modeling genetic differentiation between sampled populations or individuals may be that of isolation by distance (IBD) (Wright).  The pattern of IBD, in which genetic differentiation increases with the geographic distance between populations, is ubiquitous in nature (Meirmans 2012, Ralph \& Coop 2012), and relies only on the weak assumption that individual's mating opportunities are geographically limited by dispersal.  

%Isolation by distance is, in many ways, a more reasonable null hypothesis of population relatedness than a population phylogeny;  \gb{too compare-y?  should I ditch this?} population structure will only rarely be truly tree-like, and a strictly bifurcating graph is unable to accommodate many geographic scenarios, such as multiple equidistant populations in migration-drift equilibrium.

Here, we present an analytical framework and develop an inference algorithm for studying the spatial distribution of genetic variation based on a model of isolation by distance.  We also introduce a simple statistic for quantifying a population's admixture status, and demonstrate the utility of this approach with two high profile empirical applications.

%%%%%%%%% %%%%%%%%% %%%%%%%%%
\section*{Methods}
\subsection*{Data}
Our data consist of $L$ unlinked variable loci sampled across $K$ populations, as well as the geographic coordinates, denoted $G_k$, (latitude and longitude) at which those populations were sampled.  For convenience, we use bi-allelic single nucleotide polymorphisms (SNPs) as the genetic data in our descriptions below, but we note that this method could be applied equally well to other types of data, including microsatellites.  We refer to the $K$ accessions as populations, but the method can also be applied to individuals.  We summarize these genetic data as a set of allelic count data and sample size data, arbitrarily choosing an allele to count at each locus, and denoting the number of the counted allele at locus $\ell$ in population $k$ as $C_{\ell,k}$ out of a total sample size of $S_{\ell,k}$ alleles.  The sample frequency of the counted allele at locus $\ell$ in population $k$ is therefore $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  Our method models the covariance in these allele frequencies between populations across loci.


%%%%%%%%% %%%%%%%%% 
\subsection*{The Normal Approximation to Drift}
In order to model this covariance, we draw on previous work (Cavalli-Sforza \& Edwards, Nicholson et al 2002, Coop et al 2010, Pickrell \& Pritchard 2012), and model the process of genetic drift as approximately Gaussian.  Briefly, if the frequency of allele $\epsilon$ in an ancestral population, $A$ is $\epsilon_A$, then we model the frequency of that allele in a daughter population,  $B$, as follows:
\begin{equation}
\label{eq:normal_drift}
\epsilon_B \sim N(\epsilon_A,\delta_B(\epsilon_A)(1-\epsilon_A))
\end{equation}
The parameter $\delta_B$, the amount of drift that has occurred in $B$ since it split off from $A$, is approximately equal to $\frac{t}{2N_e}$, where $t$ is time in generations, and $N_e$ is the effective population size.  Note that the binomial variance term in the normal variance in \eqref{eq:normal_drift} ignores the fact that the drift variance will change from generation to generation due to changes in the frequency of $\epsilon$ within a population over time.  Note also that this approximation works best for alleles at intermediate frequency, which are less likely to be lost or fixed during the interval separating $A$ and $B$.  However, if the extent of drift sufficiently limited (time scales are short or population sizes are large), the approximation of inter-generational binomial sampling to Brownian motion is good (\gb{see figure comparing fit of binomial sampling to fit of normal approx?}).  

Between populations, the differentiating process of drift is counteracted by the homogenizing force of migration, so that populations with higher levels of historical or ongoing migration can be thought of as having more shared drift.  Across loci, populations with highly shared drift will tend to covary more strongly in the deviations their allele frequencies take from some ancestral (or global) mean.

%%%%%%%%% %%%%%%%%% 
\subsection*{Mean-centering and normalizing variance}
The multivariate normal distribution offers a natural statistical framework for describing this covariance, which may be straightforwardly modeled as a parametric function of any pairwise variable (see, e.g., Bradburd, Ralph, and Coop 2013).  Unlike multivariate normal observations however, allele frequencies at variable loci are constrained to have a mean between 0 and 1, and may have heterogeneous variance across loci.  To more closely meet the assumptions of our statistical inference model, we mean-center and normalize our observations by the following procedure.

Recall that the sample allele frequency at locus $\ell$ in population $k$ is given by $\hat{f}_{\ell,k} = C_{\ell,k}/S_{\ell,k}$.  We wish to calculate a sample mean frequency at each locus weighted by the sample size in each population.  As sample size may vary across loci, we first calculate $\bar{S}_k$, the mean population sample size in population $k$, as $\bar{S}_k = \frac{1}{\ell}\sum_\ell S_{\ell,k}$.  We then calculate the weighted sample mean frequency at locus $\ell$ as follows:
\begin{equation}
\label{eq:sample_mean_freq}
\bar{f}_{\ell} = \frac{1}{\sum_k S_{\ell,k}} \sum_k \hat{f}_{\ell,k} S_{\ell,k}
\end{equation}
The binomial variance at each locus varies as a function of the global allele frequency, and, at locus $\ell$, is given by $\bar{f}_{\ell}(1-\bar{f}_{\ell})$.  To avoid modeling the heterogeneous effect of this variance across loci, we standardize by the binomial variance.  We call the mean-centered normalized allele frequencies $X_{\ell,k}$, and calculate them as follows:
\begin{equation}
\label{eq:MCN_freqs}
X_{\ell,k} = \frac{ \hat{f}_{\ell,k} - \bar{f}_{\ell} } {\bar{f}_{\ell}(1-\bar{f}_{\ell})}
\end{equation}
We then model these transformed data as multivariate normal, as described in the next section.  Note, by using the sample mean frequency to mean-center our observations, we lose a degree of freedom, and can also induce negative covariance across loci between populations.  We discuss this further in Appendix A.

%%%%%%%%% %%%%%%%%% 
\subsection*{Spatial Covariance Model}
We wish to model the covariance between populations as the result of a spatial process, in which migration rates between nearby populations are higher than between distant ones, so that a population has higher covariance with a close neighbor than with a more distant population.  The form of our parametric covariance matrix fits a model of exponential decay to allele frequency covariance with geographic distance (Wasser et al 2004, Bradburd, Ralph, and Coop 2013).  The covariance between populations $i$ and $j$ is 
\begin{equation}
\label{eq:spatial_covariance}
\omega_{i,j} = \frac{1}{\alpha_0} \text{exp} \left(	\left( \alpha_1D_{i,j} \right)^{\alpha_2} \right)\text{,}
\end{equation}
where $D_{i,j}$ is the geographic distance between population $i$ and $j$, $\alpha_0$ controls the sill of the covariance (the within-population variance, or the covariance when distance between points is 0),  $\alpha_1$ controls the rate of the decay of covariance per unit pairwise distance, and $\alpha_2$ determines the shape of that decay.
We model the sample covariance of mean-centered normalized allele frequencies ($Y$) as Wishart with degrees of freedom equal to the number of loci ($L$) across which the covariance is calculated.
That is, 
\begin{equation}
\label{eq:wishart_dist}
Y \sim \mathcal{W}\left( L^{-1} \omega, L	\right)
\end{equation}


The likelihood of the data $X_{\ell,k}$, given the observed distance matrix $D$ between all $K$ populations and the values of the three $\alpha$ parameters ($\vec{\alpha}$), which together parameterize the spatial covariance matrix $\omega$, is given by
\begin{equation}
\label{eq:wishart_lnl}
\mathcal{P}(X_{\ell,k} \; | \; D, \vec{\alpha}) = \frac{|Y|^{ \frac{L - K - 1}{2} } \text{exp} \left(  \frac{-\text{tr}(\omega^{-1}Y)}{2}	\right) }	
									{2^{\frac{LK}{2}}  |\frac{\omega}{L}|^{\frac{L}{2}}  \Gamma_{p}\!\left(  \frac{L}{2} \right)	},
\end{equation}
where $\text{tr}$ denotes the trace function and $\Gamma_p$ denotes a multivariate gamma function.


%%%%%%%%% %%%%%%%%% 
\subsection*{Accommodating non-equilibrium processes}
This model assumes that the system is in migration-drift equilibrium and that every unit of pairwise geographic distance is equivalent in its contribution to decay of allelic covariance, but in many cases those assumptions will not be met.

Non-equilibrium processes like long distance dispersal, recent migration, colonization, or population expansion events will distort the shape of the decay of covariance with distance, and may bias parameter estimation.  Depending on the distribution and history of these events over the landscape, they may also result in heterogeneous patterns of the decay of covariance over space.  Strong barriers to dispersal on the landscape can also bias the estimation of the contribution of a unit distance to decay in covariance.

To accommodate these heterogeneous processes, we can treat population's locations on the landscape as a random variable, and estimate them as part of our inference procedure.  Two populations that are sampled at distant locations but that are genetically similar (perhaps one was recently founded by a colonization event from the other) may choose estimated locations that are nearby, while two populations that are sampled close together, but that are genetically dissimilar (e.g, the recently founded population and its neighbor), may choose locations that are farther apart.
The result is a map \gb{how do we feel about the word "geogenetic?"} in which the distances between populations are indicative of the way that populations perceive the distances between themselves.  There is a rich history of this type of data visualization, especially using reduced dimensionality representations of the data such as Principal Components Analysis (e.g. Menozzi, Piazza, and Cavalli-Sforza 1978, Novembre et al 2008, Novembre and Stephens 2008).  Our method offers an improvement over PCA-based visualization methods in that it offers an explicit model of spatial genetic variation, and can therefore be used as a robust framework for inference and hypothesis testing (see also Yang et al 2012, Yang et al 2014).

Our likelihood function is now XXXXX.   (Spatial priors?)

We can initiate from random locations to remove any influence of the observed map on the output via the prior, or we can change the variance on the spatial priors to ascertain the effect of the prior on inference.

%%%%%%%%% %%%%%%%%% 
\subsection*{Spatial Admixture Statistic}
Given a model of isolation by distance and allele frequencies sampled at different points on a map, we can calculate (1) whether a population's genetic makeup is well predicted by that of its neighbors, and (2) whether a focal population has excess covariance with another sampled population.

%%%%%%%%% %%%%%%%%% 
\subsection*{Inference of Spatial Admixture}
We can interpret this excess covariance over anomalously long distances as genetic admixture, and we can incorporate it into out inference framework.  We define the allele frequency at locus $\ell$ in admixed population $k$ to be
\begin{equation}
f_{\ell,k} = pf_{\ell,i} + (1-p)f_{\ell,j},
\end{equation}
where $f_{\ell,i}$ is the allele frequency in population $i$, $f_{\ell,j}$ is the allele frequency in population $j$, and $p$ is the admixture proportion, which varies between 0 and 1 and describes the extent to which populations $i$ and $j$ are contributing to the genetic make-up of population $k$.

To infer the spatial context of this admixture, we allow each population a point in space (\gb{can I use the phrase `ghost population,' or is that too goofy?}) from which it draws its admixture, and we model both the location of that point and the extent (proportion) of that admixture.  The observed allele frequencies in sampled populations are therefore a weighted average of the model-estimated allele frequencies at the geographic location of the sampled population and those at the coordinates from which the observed population draws admixture.  That is, the observed allele frequencies in population $k$ are modeled as follows:
\begin{equation}
f_{k} = pf_{k'} + (1-p)f_{j},
\end{equation}
where $f_{k'}$ are the model-estimated allele frequencies across loci at the spatial location of population $k$ and $f_{j}$ are the model-estimated allele frequencies at the spatial location $j$ from which population $k$ is drawing admixture in proportion $p$.  The admixture proportion $p$ is constrained to vary between 0 and 0.5, such that at least half of a population's genetic make-up must be determined by its geographic location.

The covariance between sampled populations $i$ and $j$ can be modeled as
\begin{alignat}{3}
\label{eq:admixed_covariance_1}
\Omega_{i,j} = (1-p_i)(1-p_j) \omega_{i\;,\;j\;} \; \times&\\
(p_i)(1-p_j) \omega_{i',\;j\;} \; \times   \notag&\\
(p_j)(1-p_i) \omega_{i\;,\;j'} \; \times   \notag&\\
(p_i)(p_j) \omega_{i',\;j'} \; \phantom{\times}   \notag&
\end{alignat}
where $i'$ and $j'$ are the locations from which populations $i$ and $j$ are drawing their admixture with proportions $p_i$ and $p_j$, and $\omega$ is the spatial covariance function parameterized by the pairwise geographic distances between each pair of populations.
In addition, we introduce a population-specific variance term on each diagonal element of the covariance matrix.  This term accommodates both the binomial sampling noise introduced by finite chromosome sample size in each population and also population-specific drift.  The full expression for our admixed covariance function is below.
\begin{alignat}{3}
\label{eq:admixed_covariance_2}
\Omega_{i,j} = (1-p_i)(1-p_j) \omega_{i\;,\;j\;} \; \times&\\
(p_i)(1-p_j) \omega_{i',\;j\;} \; \times   \notag&\\
(p_j)(1-p_i) \omega_{i\;,\;j'} \; \times   \notag&\\
(p_i)(p_j) \omega_{i',\;j'} \; +   \notag&\\
I\bar{S_k}^{-1} + \eta_k \phantom{+} \notag&
\end{alignat}

where $I$ is the identity matrix, $\bar{S_k}$ is the mean sample size in population $k$ across all loci, and $\eta_k$ is the nugget estimated in population $k$.

The locations of the set of $k$ points from which the $k$ sampled populations draw admixture, and the vector of length $k$ of each population's admixture proportion, $p_k$, are treated as random variables and jointly estimated as part of our inference procedure.

The likelihood function for this model in which each population is allowed to draw admixture from a point in space is therefore XXXX.

For details on our Bayesian inference framework and Markov chain Monte Carlo inference procedure, please see the Section: How I spent the past year!

\subsection*{Models}
The models described above may be used in various combinations.  In the simplest model, populations may not choose their own locations, nor are they allowed to draw admixture, and the only parameters to be estimated are those of the spatial covariance function given in Eqn \eqref{eq:spatial_covariance}, and the population-specific variance terms ($\eta_k$).  In the most complex model, population locations, the locations of their sources of admixture, and the proportions of that admixture are all estimated jointly in addition to the parameters of the spatial covariance function and the population specific variances.  %The other combinations involve keeping population locations fixed, but allowing them to draw admixture, and allowing populations to move, but not allowing them to draw admixture.

\gb{A note about here how this complex model is still identifiable}

\section*{Empirical Applications}
To demonstrate the applications of this novel method, we employed it in two canonical empirical systems: the greenish warbler ring species complex, and, to our knowledge, the most complete geographic sampling of human populations to date (\gb{is this true?}).

\subsection*{Greenish Warblers}
Description of the data

Description of the research questions we use SpaceMix to address

Details of analyses run
\subsection*{Human Populations}
Description of the data

Description of the research questions we use SpaceMix to address

Details of analyses run

%%%%%%%%% %%%%%%%%% 
\section*{Results}
\gb{should we just merge the results section in with the empirical applications section?  not sure what else would go here.}

%%%%%%%%% %%%%%%%%% 
\section*{Discussion}
In this paper we have presented blah blah blah.  We believe this represents an advance over previous methods because blah blah blah.  This method can be used to answer a variety of empirical questions, including blah, blah, and blah, and also serves as an intuitive data visualization tool.

%%%%%%%%% %%%%%%%%% 
\subsection*{Empirical Results}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Greenish Warblers}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\subsubsection*{Humans}

\gb{not sure how much to include here}

%%%%%%%%% %%%%%%%%% 
\section*{Future Directions}

spatiotemporal model

spatialSTRUCTURE
%%%%%%%%%





\end{document}













